Index: apps/konqueror/src/konqmainwindow.cpp
===================================================================
--- apps/konqueror/src/konqmainwindow.cpp	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konqueror/src/konqmainwindow.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -3614,7 +3614,7 @@
   m_paRemoveView->setIcon( KIcon("view-close") );
   m_paRemoveView->setText( i18n( "&Close Active View" ) );
   connect(m_paRemoveView, SIGNAL(triggered()), SLOT( slotRemoveView() ));
-  m_paRemoveView->setShortcut(Qt::CTRL+Qt::SHIFT+Qt::Key_R);
+  m_paRemoveView->setShortcut(Qt::CTRL+Qt::SHIFT+Qt::Key_W);
   m_paRemoveTab = actionCollection()->addAction("removecurrenttab");
   m_paRemoveTab->setIcon( KIcon("tab-close") );
   m_paRemoveTab->setText( i18n( "Close Current Tab" ) );
@@ -5227,10 +5227,10 @@
 //
 
 // prepend http://www. or http:// if there's no protocol in 's'
-// used only when there are no completion matches
+// This is used only when there are no completion matches
 static QString hp_tryPrepend( const QString& s )
 {
-    if( s.isEmpty() || s[ 0 ] == '/' )
+    if (s.isEmpty() || s[0] == QLatin1Char('/') || s[0] == QLatin1Char('~'))
         return QString();
     for( int pos = 0;
          pos < s.length() - 2; // 4 = ://x
Index: apps/konqueror/settings/kio/proxy.desktop
===================================================================
--- apps/konqueror/settings/kio/proxy.desktop	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konqueror/settings/kio/proxy.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -123,7 +123,7 @@
 Comment[el]=Ρυθμίστε τους διαμεσολαβητές που χρησιμοποιούνται
 Comment[en_GB]=Configure the proxy servers used
 Comment[eo]=Agordi la prokurilon
-Comment[es]=Configuración de los servidores Proxy usados
+Comment[es]=Configuración de los servidores sroxy usados
 Comment[et]=Kasutatavate puhverserverite seadistamine
 Comment[eu]=Konfiguratu erabilitako proxy zerbitzariak
 Comment[fa]=پیکربندی پیشکارهای استفاده‌شده
Index: apps/konqueror/settings/kio/bookmarks.desktop
===================================================================
--- apps/konqueror/settings/kio/bookmarks.desktop	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konqueror/settings/kio/bookmarks.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -114,7 +114,7 @@
 Comment[el]=Ρύθμιση της αρχικής σελίδας των σελιδοδεικτών
 Comment[en_GB]=Configure the bookmarks home page
 Comment[eo]=Agordi la ĉefpaĝon de la legosignoj
-Comment[es]=Configuración de los marcadores
+Comment[es]=Configuración de la página inicial de los marcadores
 Comment[et]=Järjehoidjate kodulehekülje seadistamine
 Comment[eu]=Konfiguratu laster-marken hasierako orria
 Comment[fi]=Muokkaa kirjanmerkkien kotisivua
Index: apps/konqueror/settings/kio/cache.desktop
===================================================================
--- apps/konqueror/settings/kio/cache.desktop	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konqueror/settings/kio/cache.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -120,7 +120,7 @@
 Comment[el]=Ρυθμίστε τη λανθάνουσα μνήμη ιστοσελίδων
 Comment[en_GB]=Configure web cache settings
 Comment[eo]=Agordi la retan tenejon
-Comment[es]=Configurar las preferencias del caché de web
+Comment[es]=Configuración de las preferencias de la caché web
 Comment[et]=Veebi vahemälu seadistamine
 Comment[eu]=Konfiguratu web katxearen ezarpenak
 Comment[fa]=پیکربندی تنظیمات نهانگاه وب
Index: apps/konqueror/settings/kio/netpref.desktop
===================================================================
--- apps/konqueror/settings/kio/netpref.desktop	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konqueror/settings/kio/netpref.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -118,7 +118,7 @@
 Comment[el]=Ρύθμιση γενικών προτιμήσεων δικτύου, όπως τιμές χρονικών ορίων
 Comment[en_GB]=Configure generic network preferences, like timeout values
 Comment[eo]=Agordi la ĝeneralajn retopciojn, kiel tempolimojn ktp.
-Comment[es]=Configuración de las preferencias genéricas de red, como los tiempos límite
+Comment[es]=Configuración de las preferencias genéricas de red, como los tiempos de espera límite
 Comment[et]=Üldised võrguseadistused
 Comment[eu]=Konfiguratu sarerako hobespen orokorrak (denbora-muga balioak, besteak beste)
 Comment[fa]=پیکربندی تنظیمات شبکۀ محلی، مانند مقادیر اتمام‌ وقت
Index: apps/konqueror/settings/konqhtml/khtml_behavior.desktop
===================================================================
--- apps/konqueror/settings/konqhtml/khtml_behavior.desktop	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konqueror/settings/konqhtml/khtml_behavior.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -29,7 +29,7 @@
 Name[el]=Συμπεριφορά ιστού
 Name[en_GB]=Web Behaviour
 Name[eo]=Reta konduto
-Name[es]=Comportamiento
+Name[es]=Comportamiento web
 Name[et]=Brauseri käitumine
 Name[eu]=Web portaera
 Name[fa]=رفتار وب
Index: apps/konqueror/settings/konqhtml/khtml_appearance.desktop
===================================================================
--- apps/konqueror/settings/konqhtml/khtml_appearance.desktop	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konqueror/settings/konqhtml/khtml_appearance.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -115,7 +115,7 @@
 Comment[el]=Ρύθμιση της εμφάνισης των ιστοσελίδων
 Comment[en_GB]=Configure how to display web pages
 Comment[eo]=Agordi la vidigilon de la TTT-paĝoj
-Comment[es]=Configuración de los tipos de letra usados en páginas web
+Comment[es]=Configuración de los tipos de letra usados para mostrar páginas web
 Comment[et]=Veebilehtede väljanägemise seadistamine
 Comment[eu]=Konfiguratu web orriak nola bistaratu
 Comment[fi]=Muokkaa sitä kuinka verkkosivut näytetään
Index: apps/konqueror/kttsplugin/khtmlkttsd.desktop
===================================================================
--- apps/konqueror/kttsplugin/khtmlkttsd.desktop	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konqueror/kttsplugin/khtmlkttsd.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -112,7 +112,7 @@
 Comment[el]=Εκφωνεί το κείμενο της τρέχουσας σελίδας
 Comment[en_GB]=Produces audio output for text in the current page
 Comment[eo]=Generas parolaĵon el la teksto en la nuna paĝo
-Comment[es]=Produce audio del texto de la página actual
+Comment[es]=Genera una salida de audio a partir del texto de la página actual
 Comment[et]=Aktiivse lehekülje teksti ettelugemine
 Comment[eu]=Testurako audio irteera sortzen du uneko orrian
 Comment[fa]=خروجی صوتی را برای متن در صفحه جاری تولید می‌کند
Index: apps/konqueror/sidebar/trees/init/remote/ftp/.directory
===================================================================
--- apps/konqueror/sidebar/trees/init/remote/ftp/.directory	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konqueror/sidebar/trees/init/remote/ftp/.directory	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -10,7 +10,7 @@
 Name[bn_IN]=FTP আর্কাইভ
 Name[ca]=Arxius FTP
 Name[ca@valencia]=Arxius FTP
-Name[cs]=FTP archívy
+Name[cs]=FTP archivy
 Name[csb]=Archiwa FTP
 Name[da]=FTP-arkiver
 Name[de]=FTP-Archive
Index: apps/konqueror/sidebar/default_entries/history.desktop
===================================================================
--- apps/konqueror/sidebar/default_entries/history.desktop	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konqueror/sidebar/default_entries/history.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -110,7 +110,7 @@
 Comment[el]=Αυτό είναι το ιστορικό των URL που επισκεφθήκατε πρόσφατα. Μπορείτε να τα ταξινομήσετε με πολλούς τρόπους.
 Comment[en_GB]=This is the history of the URLs you have recently visited. You can sort them in many ways.
 Comment[eo]=Jen la historio de la vizititaj URLoj. Vi povas ordigi ilin diversmaniere.
-Comment[es]=Este es el historial con las URLs que ha visitado recientemente. Puede ordenarlas de diversos modos.
+Comment[es]=Este es el historial con los URL que ha visitado recientemente. Puede ordenarlos de diversos modos.
 Comment[et]=Sinu viimati külastatud saitide ajalugu. Ajalugu on võimalik mitmel moel sorteerida.
 Comment[eu]=Hau bisitatu berri dituzun URLen historia da. Era askotan antola ditzakezu
 Comment[fa]=این تاریخچۀ نشانیهای وبی است که اخیراً بازدید کرده‌اید. می‌توانید آنها را به روشهای زیادی مرتب کنید.
Index: apps/dolphin/src/dolphindetailsview.cpp
===================================================================
--- apps/dolphin/src/dolphindetailsview.cpp	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/dolphin/src/dolphindetailsview.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -987,7 +987,19 @@
     setIconSize(QSize(iconSize, iconSize));
     m_decorationSize = QSize(iconSize, iconSize);
 
+    if (m_extensionsFactory) {
+        // The old maximumSize used by KFileItemDelegate is not valid any more after the icon size change.
+        // It must be discarded before doItemsLayout() is called (see bug 234600).
+        m_extensionsFactory->fileItemDelegate()->setMaximumSize(QSize());
+    }
+
     doItemsLayout();
+
+    // Calculate the new maximumSize for KFileItemDelegate after the icon size change.
+    QModelIndex current = currentIndex();
+    if (current.isValid()) {
+        adjustMaximumSizeForEditing(current);
+    }
 }
 
 KFileItemDelegate::Information DolphinDetailsView::infoForColumn(int columnIndex) const
Index: apps/doc/konqueror/bookmarks.docbook
===================================================================
--- apps/doc/konqueror/bookmarks.docbook	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/doc/konqueror/bookmarks.docbook	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -5,8 +5,8 @@
 <!-- TRANS:ROLES_OF_TRANSLATORS -->
 </authorgroup>
 
-<date>2010-02-26</date>
-<releaseinfo>&kde; 4.4</releaseinfo>
+<date>2010-09-24</date>
+<releaseinfo>&kde; 4.5</releaseinfo>
 </chapterinfo>
 
 <title>Using Bookmarks</title>
@@ -15,10 +15,9 @@
 your own files and folders, they are most useful when you are surfing the
 Web, letting you build up a list of useful sites.</para>
 
-<para>To open the <guimenu>Bookmarks</guimenu> menu you may either
+<para>To open the <guimenu>Bookmarks</guimenu> menu you may 
 <mousebutton>left</mousebutton> click on the
-<guimenu>Bookmarks</guimenu> menu or use the <keycombo
-action="simul">&Alt;<keycap>B</keycap></keycombo> shortcut. Once
+<guimenu>Bookmarks</guimenu> menu. Once
 the list is visible, you can navigate through it with the arrow keys or with
 your mouse, then press &Enter; or
 <mousebutton>left</mousebutton> click to visit the selected location.</para>
@@ -27,7 +26,7 @@
 </guimenu><guimenuitem>Add Bookmark</guimenuitem></menuchoice> or 
 <keycombo action="simul">&Ctrl;<keycap>B</keycap></keycombo> or
 <mousebutton>right</mousebutton> click on a clear space in the web page or 
-folder view and select <guimenuitem>Bookmark this Location</guimenuitem> from 
+folder view and select <guimenuitem>Bookmark this Page</guimenuitem> from 
 the pop up menu.</para>
 
 <para>The <guimenu>Bookmarks</guimenu> list can contain subfolders containing 
@@ -38,8 +37,11 @@
 the <guimenuitem>Add Bookmark</guimenuitem> item in that folder.</para>
 
 <para>You can also access your bookmarks from the <link linkend="sidebar">
-<guilabel>Sidebar</guilabel></link>.</para>
+<guilabel>Sidebar</guilabel></link> or from the <guilabel>Bookmark Toolbar</guilabel>.</para>
 
+<para>If you enter <userinput>bookmarks:/</userinput> in the location bar the 
+bookmarks home page with an overview of all bookmarks and folders is shown.</para> 
+
 <sect1 id="orgbmark">
 <title>The Bookmark Editor</title>
 
@@ -80,7 +82,7 @@
 the <guimenu>Edit</guimenu> menu or the pop up menu that appears when you 
 <mousebutton>right</mousebutton> click on the item. Similarly, you can edit 
 the &URL; by pressing <keycap>F3</keycap> or choosing the <guimenuitem>
-Change URL</guimenuitem> menu item.</para>
+Change Location</guimenuitem> menu item.</para>
 
 <para>The <guilabel>Bookmark Editor</guilabel> lets you import bookmarks from 
 a range of other browsers into &konqueror;'s bookmark list, putting them into 
@@ -92,7 +94,7 @@
 
 <para>To select which bookmark subfolder is used to hold the Bookmark 
 Toolbar items select the subfolder then choose <guimenuitem>Set as Toolbar 
-Folder</guimenuitem> from the <guimenu>Edit</guimenu> menu.</para>
+Folder</guimenuitem> from the <guimenu>Folder</guimenu> menu.</para>
 
 <para>If you are tidying up your bookmarks and have forgotten what a 
 particular web page is, you can easily open it from within the <guilabel>
Index: apps/doc/kwrite/index.docbook
===================================================================
--- apps/doc/kwrite/index.docbook	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/doc/kwrite/index.docbook	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -40,7 +40,7 @@
 </copyright>
 <legalnotice>&FDLNotice;</legalnotice>
 
-<date>2010-03-04</date>
+<date>2010-09-16</date>
 <releaseinfo>4.5</releaseinfo>
 
 <abstract>
@@ -1373,7 +1373,7 @@
 <para>You can overwrite the default encoding set in <menuchoice>
 <guimenu>Settings</guimenu>
 <guimenuitem>Configure
-Editor...</guimenuitem></menuchoice> in the <guilabel>Open/Save</guilabel> tab
+Editor...</guimenuitem></menuchoice> in the <guilabel>Open/Save</guilabel> page
 to set a different encoding for your current document. The encoding you
 set here will be only valid for your current document.</para>
  </listitem>
@@ -1395,7 +1395,7 @@
 <varlistentry>
 <term><anchor id="tools-add-byte-mark-order"/><menuchoice>
 <guimenu>Tools</guimenu>
-<guisubmenu>Add Byte Mark Order</guisubmenu></menuchoice></term>
+<guisubmenu>Add Byte Mark Order (BOM)</guisubmenu></menuchoice></term>
  <listitem>
 <para>Checking this action you can explicitly add a byte order mark for unicode
 encoded documents. The byte order mark (BOM) is a Unicode character used to signal 
@@ -1423,8 +1423,10 @@
 <guimenuitem>Word Completion</guimenuitem>
 </menuchoice> (Power user mode)</term>
 <listitem>
-<para><guimenuitem>Reuse Word Below</guimenuitem> and <guimenuitem>Reuse Word
-Above</guimenuitem> complete the currently typed text by searching for similar
+<para><guimenuitem>Reuse Word Below</guimenuitem> (<keycombo action="simul">&Ctrl;
+<keycap>9</keycap></keycombo>) and <guimenuitem>Reuse Word Above</guimenuitem> 
+(<keycombo action="simul">&Ctrl; <keycap>8</keycap></keycombo>) complete the 
+currently typed text by searching for similar
 words backward or forward from the current cursor position. <guimenuitem>Shell
 Completion</guimenuitem> pops up a completion box with matching entries.</para>
 </listitem>
@@ -1436,10 +1438,10 @@
 <keycombo action="simul">&Ctrl;&Shift;<keycap>O</keycap></keycombo>
 </shortcut>
 <guimenu>Tools</guimenu>
-<guimenuitem>Automatic Spelling</guimenuitem>
+<guimenuitem>Automatic Spell Checking</guimenuitem>
 </menuchoice></term>
 <listitem>
-<para>When <guimenuitem>Automatic Spelling</guimenuitem> is enabled, wrong text
+<para>When <guimenuitem>Automatic Spell Checking</guimenuitem> is enabled, wrong text
 is underlined in the document on-the-fly.</para>
 </listitem>
 </varlistentry>
@@ -1581,6 +1583,7 @@
  </listitem>
 </varlistentry>
 
+<!-- Indent/Inindent no longer in the menu ???, but shortcuts work-->
 <varlistentry>
 <term><anchor id="indent"/><menuchoice>
 <shortcut>
@@ -1603,10 +1606,9 @@
 <listitem>
 <para>This reduces the paragraph's indentation by one step. The size of the step
 depends on the <link linkend="pref-edit-indent"> indentation settings</link>.</para>
-
 </listitem>
 </varlistentry>
-
+<!-- -->
 <varlistentry>
 <term><anchor id="cleanindent"/><menuchoice>
 <guimenu>Tools</guimenu><guimenuitem>Clean
@@ -1682,7 +1684,7 @@
 <varlistentry>
 <term><menuchoice>
 <shortcut>
-<keycombo action="simul">&Alt;&Ctrl;<keycap>U</keycap></keycombo>
+<keycombo action="simul">&Ctrl;&Alt;<keycap>U</keycap></keycombo>
 </shortcut>
 <guimenu>Tools</guimenu>
 <guimenuitem>Capitalize</guimenuitem>
@@ -1707,7 +1709,7 @@
 <varlistentry>
 <term><menuchoice>
 <guimenu>Tools</guimenu>
-<guimenuitem>Word Wrap Document</guimenuitem>
+<guimenuitem>Apply Word Wrap</guimenuitem>
 </menuchoice></term>
 <listitem><para>Apply static word wrapping on all the document. That means that
 a new line of text will automatically start when the current
@@ -1721,6 +1723,7 @@
 <para>If the <link linkend="pref-plugins">plugin</link> Insert File is enabled, you will
 find here an additional menu item <guimenuitem>Insert File...</guimenuitem>.
 </para>
+<!-- Data Tools and Insane HTML (LE) Expansion Ctrl+. when plugins are enabled-->
 </sect1>
 
 <sect1 id="settings">
@@ -2240,7 +2243,7 @@
 
 <varlistentry>
 <term><guilabel>Allow scrolling past the end of the document</guilabel></term> <listitem>
-<para>This option lets you scroll past the end of the document. This can be used to vertically centre the bottom of the document, or put it on top of the current view.</para></listitem>
+<para>This option lets you scroll past the end of the document. This can be used to vertically center the bottom of the document, or put it on top of the current view.</para></listitem>
 </varlistentry>
 </variablelist>
 </listitem>
@@ -2540,28 +2543,49 @@
 <term><guilabel>File Format</guilabel></term>
 <listitem>
 <variablelist>
+
 <varlistentry>
 <term><guilabel>Encoding:</guilabel></term>
 <listitem>
-<para>This sets the default character encoding for your files.</para>
+<para>This defines the standard encoding to use to open/save files, if not changed 
+in the open/save dialog or by using a command line option.</para>
 </listitem>
 </varlistentry>
+
 <varlistentry>
-<term><guilabel>Encoding autodetection</guilabel></term>
+<term><guilabel>Encoding Detection</guilabel></term>
 <listitem>
 <para>Select an item from the drop down box, either to disable autodetection or
 use <guilabel>Universal</guilabel> to enable autodetection for all encodings. But as
 this may probably only detect utf-8/utf-16, selecting a region will use custom heuristics
-for better results.</para>
+for better results.
+If neither the encoding chosen as standard above, nor the encoding specified 
+in the open/save dialog, nor the encoding specified on command line match 
+the content of the file, this detection will be run.</para>
 </listitem>
 </varlistentry>
+
 <varlistentry>
+<term><guilabel>Fallback Encoding</guilabel></term>
+<listitem>
+<para>This defines the fallback encoding to try for opening files if neither the 
+encoding chosen as standard above, nor the encoding specified in the open/
+save dialog, nor the encoding specified on command line match the content of 
+the file.  Before this is used, an attempt will be made to determine the 
+encoding to use by looking for a byte order marker at start of file: if one 
+is found, the right unicode encoding will be chosen; otherwise encoding 
+detection will run, if both fail fallback encoding will be tried.</para>
+</listitem>
+</varlistentry>
+
+<varlistentry>
 <term><guilabel>End of line:</guilabel></term>
 <listitem>
 <para>Choose your preferred end of line mode for your active
 document. You have the choice between &UNIX;, DOS/&Windows; or Macintosh.</para>
 </listitem>
 </varlistentry>
+
 <varlistentry>
 <term><guilabel>Automatic end of line detection</guilabel></term>
 <listitem>
@@ -2791,7 +2815,7 @@
 If you press Enter and AutoBrace is enabled, it will insert your favorite new brace where it belongs. Looks like this:
 -->
 <varlistentry>
-<term><guilabel>DataTools</guilabel></term>
+<term><guilabel>Data Tools</guilabel></term>
 <listitem><para>Enables data tools like thesaurus and spell check (if installed).
 Data tools are only available when text is selected, or when the right mouse
 button is clicked over a word. If no data tools are offered even when text
@@ -2809,11 +2833,28 @@
 as HTML to the system clipboard.</para></listitem>
 </varlistentry>
 <varlistentry>
+<term><guilabel>IconInserter</guilabel></term>
+<listitem><para>The icon inserter plugin is only of use for &kde; developers: It adds an item 
+<guimenuitem>Insert KIcon-Code</guimenuitem> into the context menu of the editor. 
+If activated, the &kde;'s icon chooser opens (showing application icons, action icons &etc;..). 
+If you click on it, the file name without file extension will be inserted as 
+text. Useful only for setting icons via &kde;'s framework through the KIcon() class.
+</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><guilabel>Insane (not ZEN) HTML coding (light edition)</guilabel></term>
+<listitem><para>A plugin, which does zen-coding like selector completion.
+For more information see <ulink url="http://en.wikipedia.org/wiki/Zen_Coding">
+Zen Coding</ulink>.</para></listitem>
+</varlistentry>
+
+<varlistentry>
 <term><guilabel>Insert File</guilabel></term>
 <listitem><para>This plugin allows you to insert any readable file at the cursor position.
 If enabled, the <guimenu>Tools</guimenu> menu has an additional menu item
 <guimenuitem>Insert File</guimenuitem>.</para></listitem>
 </varlistentry>
+<!-- not build any longer in kde 4.5
 <varlistentry>
 <term>Python Encoding check</term>
 <listitem><para>This plugin is always enabled.
@@ -2823,6 +2864,7 @@
 current encoding to the file.
 </para></listitem>
 </varlistentry>
+-->
 </variablelist>
 <!--
 kdelibs/kate/plugins/ has additional plugins
Index: apps/doc/kfind/man-kfind.1.docbook
===================================================================
--- apps/doc/kfind/man-kfind.1.docbook	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/doc/kfind/man-kfind.1.docbook	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -7,7 +7,7 @@
 <refentryinfo>
 <title>KDE User's Manual</title>
 <author>&Lauri.Watts; &Lauri.Watts.mail;</author>
-<date>June 07, 2005</date>
+<date>2010-10-06</date>
 <productname>K Desktop Environment</productname>
 </refentryinfo>
 
@@ -54,8 +54,6 @@
 </varlistentry>
 </variablelist>
 
-<para>App options, in a variablelist</para>
-
 </refsect1>
 
 <refsect1>
@@ -69,8 +67,7 @@
 
 <refsect1>
 <title>Authors</title>
-<para>The current maintainer of &kfind; is <!-- FIXME: Who? -->.  A list of
-previous contributors is available in the complete user manual available in
+<para>A list of contributors is available in the complete user manual available in
 khelpcenter.</para>
 <para>This documentation was prepared by &Lauri.Watts;
 &Lauri.Watts.mail;</para>
Index: apps/doc/dolphin/index.docbook
===================================================================
--- apps/doc/dolphin/index.docbook	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/doc/dolphin/index.docbook	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -1024,7 +1024,24 @@
 <guisubmenu>Actions</guisubmenu> submenu of &dolphin;'s context menu which
 appears when clicking a file or folder with the &RMB;.
 </para>
+<para>
+Using the <guibutton>Download New Services</guibutton> you can fetch additional 
+service menu.
+</para>
 
+<para>
+If you have installed &dolphin;'s plugins for <guilabel>Git</guilabel> or 
+<guilabel>Subversion</guilabel> from the kdesdk module an additional group
+<guilabel>Version Control Systems</guilabel> is displayed.
+If these plugins are enabled and you enter a folder which is under version control, 
+the version state (locally changed, up to date &etc;) is indicated by icons 
+and in <guilabel>Subversion</guilabel> folders you have additional entries in the 
+context menu like commit, update &etc;
+</para>
+<para>
+&dolphin; has to be restarted to activate changes of these settings.
+</para>
+
 </sect2>
 
 <sect2 id="preferences-dialog-trash">
Index: apps/lib/konq/Templates/linkCDROM.desktop
===================================================================
--- apps/lib/konq/Templates/linkCDROM.desktop	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/lib/konq/Templates/linkCDROM.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -19,7 +19,7 @@
 Name[el]=Συσκευή CD-ROM...
 Name[en_GB]=CD-ROM Device...
 Name[eo]=KD-legilo...
-Name[es]=CD-ROM ...
+Name[es]=CD-ROM...
 Name[et]=CD-ROM seade...
 Name[eu]=CD-ROM gailua...
 Name[fi]=CD-ROM-laitteet...
Index: apps/lib/konq/favicons/favicons.desktop
===================================================================
--- apps/lib/konq/favicons/favicons.desktop	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/lib/konq/favicons/favicons.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -84,7 +84,7 @@
 Comment[et]=Veebilehekülgede ikoonide salvestamine
 Comment[eu]=Webguneen ikonoak gordetzen ditu
 Comment[fi]=Säilyttää nettisivujen kuvakkeita
-Comment[fr]=Stocker les icônes des site Internet
+Comment[fr]=Stocke les icônes des site Internet
 Comment[fy]=Bewarret website byldkaikes
 Comment[ga]=Stórálann sé deilbhín suíomh Gréasáin
 Comment[he]=מאחסן צלמיות אתרים
Index: apps/konsole/src/ViewManager.cpp
===================================================================
--- apps/konsole/src/ViewManager.cpp	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konsole/src/ViewManager.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -359,13 +359,8 @@
 
     Session* session = qobject_cast<Session*>(sender());
 
-    if ( _sessionMap[qobject_cast<TerminalDisplay*>(activeView())] == session )
-    {
-        // switch to the next view before deleting the session views to prevent flicker 
-        // occurring as a result of an interval between removing the active view and switching
-        // to the next view
-        nextView();
-    }
+    // We're using setSelectionBehaviorOnRemove(QTabBar::SelectPreviousTab)
+    // so no need to manually select next tab.
 
     Q_ASSERT(session);
 
Index: apps/konsole/src/ViewContainer.cpp
===================================================================
--- apps/konsole/src/ViewContainer.cpp	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konsole/src/ViewContainer.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -435,6 +435,7 @@
     _tabBar->setDrawBase(true);
     _tabBar->setDocumentMode(true);
     _tabBar->setFocusPolicy(Qt::NoFocus);
+    _tabBar->setSelectionBehaviorOnRemove(QTabBar::SelectPreviousTab);
 
     _newTabButton = new QToolButton(_containerWidget);
     _newTabButton->setIcon(KIcon("tab-new"));
Index: apps/konsole/src/main.cpp
===================================================================
--- apps/konsole/src/main.cpp	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/konsole/src/main.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -30,7 +30,7 @@
 #include <KCmdLineArgs>
 #include <KLocale>
 
-#define KONSOLE_VERSION "2.5.2"
+#define KONSOLE_VERSION "2.5.3"
 
 using namespace Konsole;
 
Index: apps/plasma/applets/folderview/folderview.cpp
===================================================================
--- apps/plasma/applets/folderview/folderview.cpp	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/plasma/applets/folderview/folderview.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -360,7 +360,7 @@
     m_model->setMimeTypeFilterList(m_filterFilesMimeList);
     m_model->setFileNameFilter(m_filterFiles);
     m_model->setSortDirectoriesFirst(m_sortDirsFirst);
-    m_model->setDynamicSortFilter(true);
+    m_model->setDynamicSortFilter(m_sortColumn != -1);
     m_model->sort(m_sortColumn != -1 ? m_sortColumn : KDirModel::Name, Qt::AscendingOrder);
 
     DirLister *lister = new DirLister(this);
@@ -712,8 +712,10 @@
         if (m_sortColumn != -1) {
             m_model->invalidate();
             m_model->sort(m_sortColumn, Qt::AscendingOrder);
+            m_model->setDynamicSortFilter(true);
         } else if (m_iconView) {
             m_iconView->setCustomLayout(true);
+            m_model->setDynamicSortFilter(false);
         }
         updateSortActionsState();
         cg.writeEntry("sortColumn", m_sortColumn);
@@ -1583,6 +1585,7 @@
     if (column != m_sortColumn) {
         m_model->invalidate();
         m_model->sort(column, Qt::AscendingOrder);
+        m_model->setDynamicSortFilter(true);
         m_sortColumn = column;
         config().writeEntry("sortColumn", m_sortColumn);
         emit configNeedsSaving();
@@ -1683,6 +1686,7 @@
     // If the user has rearranged the icons, the view is no longer sorted
     if (m_sortColumn != -1) {
         m_sortColumn = -1;
+        m_model->setDynamicSortFilter(false);
         updateSortActionsState();
         config().writeEntry("sortColumn", m_sortColumn);
         emit configNeedsSaving();
Index: apps/plasma/applets/folderview/plasma-applet-folderview.desktop
===================================================================
--- apps/plasma/applets/folderview/plasma-applet-folderview.desktop	(.../tags/KDE/4.5.2/kdebase)	(wersja 1188166)
+++ apps/plasma/applets/folderview/plasma-applet-folderview.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1188166)
@@ -91,7 +91,7 @@
 Comment[el]=Εμφάνιση περιεχομένων φακέλων (προκαθορισμένα ο προσωπικός φάκελος)
 Comment[en_GB]=Display the contents of folders (User's home folder as default)
 Comment[eo]=Montri enhavon de la dosierujo (hejma dosierujo de la uzanto kiel defaŭlta)
-Comment[es]=Mostrar el contenido de las carpetas (Carpeta de usuario como predeterminado)
+Comment[es]=Mostrar el contenido de las carpetas (carpeta personal como predeterminado)
 Comment[et]=Kataloogide sisu näitamine (vaikimisi kasutaja kodukataloog)
 Comment[eu]=Bistaratu karpeten edukia (Erabiltzaileen etxe karpeta lehenetsi gisa)
 Comment[fi]=Näytä kansion sisällön (Oletuksena työpöytäkansio)
Index: runtime/knotify/knotify4.desktop
===================================================================
Index: runtime/solid-device-automounter/kded/device_automounter.desktop
===================================================================
Index: runtime/kcontrol/componentchooser/componentchooser.desktop
===================================================================
Index: runtime/kcontrol/componentchooser/componentservices/kcm_kemail.desktop
===================================================================
Index: runtime/kcontrol/componentchooser/componentservices/kcm_browser.desktop
===================================================================
Index: runtime/kcontrol/componentchooser/componentservices/kcm_terminal.desktop
===================================================================
Index: runtime/phonon/platform_kde/phononbackend.desktop
===================================================================
Index: runtime/phonon/kded-module/phononserver.desktop
===================================================================
Index: runtime/khelpcenter/glossary.cpp
===================================================================
Index: runtime/doc/glossary/kdeprintingglossary.docbook
===================================================================
Index: runtime/doc/glossary/checkxrefs
===================================================================
Index: runtime/doc/glossary/index.docbook
===================================================================
Index: runtime/doc/kcontrol/bookmarks/index.docbook
===================================================================
Index: runtime/doc/kcontrol/emoticons/index.docbook
===================================================================
Index: runtime/doc/kcontrol/kcmnotify/index.docbook
===================================================================
Index: runtime/doc/kcontrol/attica/index.docbook
===================================================================
Index: runtime/doc/kcontrol/icons/index.docbook
===================================================================
Index: runtime/doc/kcontrol/spellchecking/index.docbook
===================================================================
Index: runtime/doc/kdesu/man-kdesu.1.docbook
===================================================================
Index: runtime/doc/kdesu/index.docbook
===================================================================
Index: runtime/doc/kioslave/nepomuksearch/index.docbook
===================================================================
Index: runtime/doc/kioslave/nepomuksearch/CMakeLists.txt
===================================================================
Index: runtime/doc/kioslave/CMakeLists.txt
===================================================================
Index: runtime/platforms/win/kwinstartmenu/misc.h
===================================================================
Index: runtime/platforms/win/kwinstartmenu/winstartmenu.cpp
===================================================================
Index: runtime/platforms/win/kwinstartmenu/linkfile.h
===================================================================
Index: runtime/platforms/win/kwinstartmenu/main.cpp
===================================================================
Index: runtime/platforms/win/kwinstartmenu/winstartmenu.h
===================================================================
Index: runtime/platforms/win/kwinstartmenu/misc.cpp
===================================================================
Index: runtime/platforms/win/kwinstartmenu/linkfile.cpp
===================================================================
Index: runtime/ktimezoned/ktimezoned.desktop
===================================================================
Index: runtime/kioslave/thumbnail/imagethumbnail.desktop
===================================================================
Index: runtime/kioslave/network/kded/networkwatcher.desktop
===================================================================
Index: runtime/kioslave/remote/kdedmodule/remotedirnotify.desktop
===================================================================
Index: runtime/kioslave/cgi/kcmcgi/kcmcgi.desktop
===================================================================
Index: runtime/kioslave/desktop/desktopnotifier.desktop
===================================================================
Index: runtime/kioslave/sftp/kio_sftp.cpp
===================================================================
Index: runtime/nepomuk/kioslaves/search/nepomuksearch.protocol
===================================================================
Index: runtime/nepomuk/services/removablestorage/nepomukremovablestorageservice.desktop
===================================================================
Index: runtime/nepomuk/services/queryservice/nepomukqueryservice.desktop
===================================================================
Index: runtime/solidautoeject/solidautoeject.desktop
===================================================================
Index: runtime/kurifilter-plugins/shorturi/kshorturifilter.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/foldoc.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/opendesktop.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/de2en.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/uspto.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/ruby_application_archive.desktop
===================================================================
Index: runtime/soliduiserver/soliduiserver.desktop
===================================================================
Index: runtime/plasma/remotewidgetshelper/kcm_remotewidgets.actions
===================================================================
Index: runtime/plasma/scriptengines/javascript/simplebindings/qscriptbookkeeping.cpp
===================================================================
Index: workspace/kcontrol/kfontinst/kcmfontinst/fontinst.desktop
===================================================================
Index: workspace/kcontrol/keyboard/keyboard_applet.h
===================================================================
Index: workspace/kcontrol/keyboard/keyboard_daemon.cpp
===================================================================
Index: workspace/kcontrol/keyboard/keyboard_applet.cpp
===================================================================
Index: workspace/kcontrol/access/kaccess.notifyrc
===================================================================
Index: workspace/doc/plasma-desktop/tools/man-plasmaengineexplorer.1.docbook
===================================================================
Index: workspace/doc/kmenuedit/index.docbook
===================================================================
Index: workspace/doc/kcontrol/screensaver/index.docbook
===================================================================
Index: workspace/doc/kcontrol/powerdevil/index.docbook
===================================================================
Index: workspace/doc/kcontrol/splashscreen/index.docbook
===================================================================
Index: workspace/doc/kcontrol/kcmsmserver/index.docbook
===================================================================
Index: workspace/doc/kcontrol/desktopthemedetails/index.docbook
===================================================================
Index: workspace/doc/kcontrol/kwintabbox/index.docbook
===================================================================
Index: workspace/doc/kcontrol/keys/index.docbook
===================================================================
Index: workspace/systemsettings/classic/settings-classic-view.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-display.desktop
===================================================================
Index: workspace/systemsettings/core/systemsettingsexternalapp.desktop
===================================================================
Index: workspace/systemsettings/icons/settings-icon-view.desktop
===================================================================
Index: workspace/kwrited/kwrited.desktop
===================================================================
Index: workspace/kwrited/kwrited-autostart.desktop
===================================================================
Index: workspace/solid/solid-actions-kcm/device-actions/solid-device-Battery.desktop
===================================================================
Index: workspace/solid/solid-actions-kcm/device-actions/solid-device-NetworkInterface.desktop
===================================================================
Index: workspace/solid/solid-actions-kcm/device-actions/solid-device-AcAdapter.desktop
===================================================================
Index: workspace/solid/solid-actions-kcm/device-actions/solid-device-SerialInterface.desktop
===================================================================
Index: workspace/powerdevil/daemon/powerdevil.desktop
===================================================================
Index: workspace/powerdevil/powerdevil.notifyrc
===================================================================
Index: workspace/libs/plasmaclock/calendartable.cpp
===================================================================
Index: workspace/libs/kdm/kgreet_classic.cpp
===================================================================
Index: workspace/libs/kdm/kgreet_winbind.cpp
===================================================================
Index: workspace/freespacenotifier/freespacenotifier.desktop
===================================================================
Index: workspace/freespacenotifier/freespacenotifier.notifyrc
===================================================================
Index: workspace/kstyles/themes/light-v2.themerc
===================================================================
Index: workspace/kstyles/themes/light-v3.themerc
===================================================================
Index: workspace/kstyles/oxygen/oxygenstyle.cpp
===================================================================
Index: workspace/khotkeys/app/khotkeys.desktop
===================================================================
Index: workspace/khotkeys/data/konqueror_gestures_kde321.khotkeys
===================================================================
Index: workspace/khotkeys/data/defaults.khotkeys
===================================================================
Index: workspace/khotkeys/data/kde32b1.khotkeys
===================================================================
Index: workspace/khotkeys/data/printscreen.khotkeys
===================================================================
Index: workspace/kinfocenter/Categories/deviceinfocategory.desktop
===================================================================
Index: workspace/kinfocenter/Categories/networkinfocategory.desktop
===================================================================
Index: workspace/kinfocenter/Categories/graphicalinfocategory.desktop
===================================================================
Index: workspace/kinfocenter/Categories/kinfocentercategory.desktop
===================================================================
Index: workspace/kinfocenter/Modules/usbview/kcmusb.desktop
===================================================================
Index: workspace/kinfocenter/Modules/view1394/kcmview1394.desktop
===================================================================
Index: workspace/kwin/effects/desktopgrid/desktopgrid.cpp
===================================================================
Index: workspace/kwin/effects/slidingpopups/slidingpopups.desktop
===================================================================
Index: workspace/kwin/kwin.notifyrc
===================================================================
Index: workspace/kdm/kfrontend/pics/stripes.png.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/ratpoison.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/aewm++.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/amaterus.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/wmaker.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/lxde.desktop
===================================================================
Index: workspace/kdm/kfrontend/kgapp.cpp
===================================================================
Index: workspace/kdm/kfrontend/kgverify.cpp
===================================================================
Index: workspace/plasma/netbook/containments/netpanel/plasma-containment-netpanel.desktop
===================================================================
Index: workspace/plasma/netbook/dataengines/searchlaunch/plasma-engine-searchlaunch.desktop
===================================================================
Index: workspace/plasma/desktop/containments/panel/plasma-containment-panel.desktop
===================================================================
Index: workspace/plasma/desktop/shell/main.cpp
===================================================================
Index: workspace/plasma/desktop/applets/kickoff/applet/plasma-applet-launcher.desktop
===================================================================
Index: workspace/plasma/desktop/applets/pager/plasma-pager-default.desktop
===================================================================
Index: workspace/plasma/desktop/runners/plasma-desktop/plasma-runner-plasma-desktop.desktop
===================================================================
Index: workspace/plasma/generic/dataengines/nowplaying/playerinterface/mpris/mpris.cpp
===================================================================
Index: workspace/plasma/generic/dataengines/weather/plasma-dataengine-weather.desktop
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/CMakeLists.txt
===================================================================
Index: workspace/plasma/generic/dataengines/geolocation/CMakeLists.txt
===================================================================
Index: workspace/plasma/generic/applets/devicenotifier/test-predicate-openinwindow.desktop
===================================================================
