Index: apps/konqueror/src/konqmainwindow.cpp
===================================================================
--- apps/konqueror/src/konqmainwindow.cpp	(.../tags/KDE/4.5.1/kdebase)	(wersja 1170567)
+++ apps/konqueror/src/konqmainwindow.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1170567)
@@ -1674,7 +1674,10 @@
       // Reuse current servicetype for local files, but not for remote files (it could have changed, e.g. over HTTP)
       QString serviceType = reloadView->url().isLocalFile() ? reloadView->serviceType() : QString();
       // By using locationBarURL instead of url, we preserve name filters (#54687)
-      openUrl( reloadView, reloadView->locationBarURL(), serviceType, req );
+      KUrl reloadUrl = reloadView->locationBarURL();
+      if (reloadUrl.isEmpty()) // e.g. initial screen
+          reloadUrl = reloadView->url();
+      openUrl(reloadView, reloadUrl, serviceType, req);
   }
 }
 
Index: apps/konqueror/settings/kio/kcookiespolicies.cpp
===================================================================
--- apps/konqueror/settings/kio/kcookiespolicies.cpp	(.../tags/KDE/4.5.1/kdebase)	(wersja 1170567)
+++ apps/konqueror/settings/kio/kcookiespolicies.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1170567)
@@ -270,6 +270,7 @@
     if (!nextItem)
       nextItem = dlg->lvDomainPolicy->itemAbove(item);
 
+    m_pDomainPolicy.remove(item);
     delete item;
   }
 
Index: apps/konqueror/settings/konqhtml/filteropts.cpp
===================================================================
--- apps/konqueror/settings/konqhtml/filteropts.cpp	(.../tags/KDE/4.5.1/kdebase)	(wersja 1170567)
+++ apps/konqueror/settings/konqhtml/filteropts.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1170567)
@@ -486,6 +486,7 @@
 
         mFilters << filterConfig;
     }
+    reset();
 }
 
 void AutomaticFilterModel::save(KConfigGroup &cg)
Index: runtime/attica/kdeplugin/kdeplatformdependent.cpp
===================================================================
Index: workspace/kcontrol/dateandtime/helper.cpp
===================================================================
Index: workspace/kcontrol/keyboard/keyboard_hardware.cpp
===================================================================
Index: workspace/kcontrol/keyboard/xkb_helper.h
===================================================================
Index: workspace/kcontrol/keyboard/xkb_helper.cpp
===================================================================
Index: workspace/kcontrol/keyboard/xinput_helper.cpp
===================================================================
Index: workspace/kcontrol/keyboard/kcm_keyboard_widget.cpp
===================================================================
Index: workspace/kinfocenter/Categories/deviceinfocategory.desktop
===================================================================
Index: workspace/kinfocenter/Categories/networkinfocategory.desktop
===================================================================
Index: workspace/kinfocenter/Categories/graphicalinfocategory.desktop
===================================================================
Index: workspace/kinfocenter/Categories/kinfocentercategory.desktop
===================================================================
Index: workspace/kwin/effects/desktopgrid/desktopgrid.cpp
===================================================================
Index: workspace/kwin/effects/blur/blur.cpp
===================================================================
Index: workspace/kwin/effects/dialogparent/dialogparent.desktop
===================================================================
Index: workspace/kwin/clients/aurorae/src/lib/auroraetab.cpp
===================================================================
Index: workspace/kwin/clients/aurorae/src/lib/auroraescene.cpp
===================================================================
Index: workspace/kwin/tiling.cpp
===================================================================
Index: workspace/plasma/desktop/shell/panelview.h
===================================================================
Index: workspace/plasma/desktop/shell/plasmaapp.h
===================================================================
Index: workspace/plasma/desktop/shell/panelview.cpp
===================================================================
Index: workspace/plasma/desktop/shell/plasmaapp.cpp
===================================================================
Index: workspace/plasma/generic/dataengines/weather/weatherengine.cpp
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/ion.cpp
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/wetter.com/ion_wettercom.cpp
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/wetter.com/ion_wettercom.h
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/envcan/ion_envcan.cpp
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/envcan/ion_envcan.h
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/ion.h
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/bbcukmet/ion_bbcukmet.cpp
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/bbcukmet/ion_bbcukmet.h
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/noaa/ion_noaa.cpp
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/noaa/ion_noaa.h
===================================================================
Index: workspace/plasma/generic/dataengines/weather/weatherengine.h
===================================================================
Index: workspace/plasma/generic/applets/battery/battery.h
===================================================================
Index: workspace/plasma/generic/applets/battery/battery.cpp
===================================================================
Index: workspace/plasma/generic/runners/kill/killrunner.cpp
===================================================================
