Index: apps/kappfinder/apps/Multimedia/alevt.desktop
===================================================================
--- apps/kappfinder/apps/Multimedia/alevt.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/kappfinder/apps/Multimedia/alevt.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -18,6 +18,7 @@
 GenericName[fi]=Videotext-näytin
 GenericName[fr]=Afficheur vidéotexte
 GenericName[fy]=Teletekst-werjefteprogramma
+GenericName[ga]=Amharcán FísTéacs
 GenericName[gl]=Visor de Videotext
 GenericName[gu]=વિડીઓટેકસ્ટ દર્શક
 GenericName[he]=מציג Videotext
@@ -31,6 +32,7 @@
 GenericName[kn]=ದೃಷ್ಯಾಂಶ (ವಿಡಿಯೋ) ಪಠ್ಯ  ಪ್ರದರ್ಶಕ
 GenericName[ko]=Videotext 뷰어
 GenericName[ku]=Nîşandêra Videotext
+GenericName[lt]=videoteksto žiūriklis
 GenericName[lv]=Videotext skatītājs
 GenericName[mk]=Прегледувач на видеотекст
 GenericName[ml]=വീഡിയോടെക്സ്റ്റ് ദര്‍ശിനി 
Index: apps/kappfinder/apps/Games/Arcade/alephone.desktop
===================================================================
--- apps/kappfinder/apps/Games/Arcade/alephone.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/kappfinder/apps/Games/Arcade/alephone.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -90,6 +90,7 @@
 Comment[fi]=Avoimen lähdekoodin versio Marathon Infinity -pelistä SDL-kirjastolla ilman Open GL -tukea
 Comment[fr]=Une version libre de Marathon Infinity pour SDL et n'utilisant pas OpenGL
 Comment[fy]=In Iepen Boarne-ferzje fan Marathon Infinity foar SDL, mar dan sûnder OpenGL
+Comment[ga]=Leagan saorbhogearra de Marathon Infinity le haghaidh SDL a bhfuil OpenGL díchumasaithe ann
 Comment[gl]=Unha versión de código aberto de Marathon Infinity para SDL con OpenGL desactivado
 Comment[gu]=SDL માટેની મેરેથોન ઇન્ફિનિટીની ઓપન સોર્સ આવૃત્તિ જેમાં OpenGL નિષ્ક્રિય કરવામાં આવ્યું છે
 Comment[he]=גרסת קוד פתוח של Marathon Infinity עבור SDL עם Open GL מבוטל
@@ -103,6 +104,7 @@
 Comment[kn]=ಎಸ್ಡಿಎಲ್ ನ ಮಾರಾಥಾನ್ ಇನ್ಫಿನಿಟಿ ಗೆ  ಓಪನ್ ಜೆಎಲ್ ಅಶಕ್ತಗೊಳಿಸಲಾದ ಒಂದು ತೆರೆದಾಕರ ಆವೃತ್ತಿ
 Comment[ko]=Marathon Infinity의 OpenGL을 사용하지 않는 SDL용 오픈소스 버전
 Comment[ku]=Guhertoya Marathon Infinity ji bo SDL'yê ya çavkaniya vekirî ya desteka wê ya Open GL neçalak bûye
+Comment[lt]=Marathon Infinity, skirto SDL, atviro kodo versija su išjungtu Open GL
 Comment[lv]=Atvērtā koda versija Marathon Infinity spēlei izmantojot SDL ar atslēgtu OpenGL
 Comment[mk]=Верзија на Marathon Infinity со отворен код за SDL која нема поддршка за Open GL
 Comment[ml]=എസ്ഡിഎല്ലിനുള്ള ഓപ്പണ്‍ജിഎല്‍ അപ്രാവര്‍ത്തികമാക്കിയിട്ടുള്ള മാരത്തോണ്‍ ഇന്‍ഫിനിറ്റിയുടെ ഓപ്പണ്‍ സോഴ്സ് പതിപ്പു്
Index: apps/kappfinder/apps/Games/TacticStrategy/freecivserver.desktop
===================================================================
--- apps/kappfinder/apps/Games/TacticStrategy/freecivserver.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/kappfinder/apps/Games/TacticStrategy/freecivserver.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -17,6 +17,7 @@
 Name[fi]=Freeciv-palvelin
 Name[fr]=Serveur FreeCiv
 Name[fy]=FreeCiv-tsjinner
+Name[ga]=Freastalaí Freeciv
 Name[gl]=Servidor de Freeciv
 Name[gu]=ફ્રીસિવ સર્વર
 Name[he]=שרת Freeciv
@@ -78,6 +79,7 @@
 Comment[fi]=Palvelin FreeCiv-pelille
 Comment[fr]=Un serveur pour FreeCiv
 Comment[fy]=In tsjinner foar FreeCiv
+Comment[ga]=Freastalaí don chluiche Freeciv
 Comment[gl]=Un servidor de Freeciv
 Comment[gu]=ફ્રીસિવ માટે સર્વર
 Comment[he]=שרת ל־Freeciv
Index: apps/konqueror/settings/kio/bookmarks.desktop
===================================================================
--- apps/konqueror/settings/kio/bookmarks.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/konqueror/settings/kio/bookmarks.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -109,6 +109,7 @@
 Comment[eu]=Konfiguratu laster-marken hasierako orria
 Comment[fr]=Configure la page d'accueil des favoris
 Comment[fy]=De wepbagina's foar blêdwizers ynstelle
+Comment[ga]=Cumraigh leathanach baile na leabharmharcanna
 Comment[gl]=Configura a páxina de inicio dos marcadores
 Comment[gu]=બુકમાર્ક્સ મુખપૃષ્ઠને રૂપરેખાંકિત કરો
 Comment[he]=הגדר את עמוד־הבית של הסימניות
@@ -134,6 +135,7 @@
 Comment[pt]=Configurar a página inicial dos favoritos
 Comment[pt_BR]=Configurar a página inicial dos favoritos
 Comment[ro]=Configurează pagina de pornire a semnelor de carte
+Comment[ru]=Настройка страницы закладок
 Comment[sk]=Nastavenie domovskej stránky záložiek
 Comment[sl]=Nastavite domačo stran z zaznamki
 Comment[sr]=Подешавање домаће странице маркера
Index: apps/konqueror/settings/konqhtml/appearance.cpp
===================================================================
--- apps/konqueror/settings/konqhtml/appearance.cpp	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/konqueror/settings/konqhtml/appearance.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -44,6 +44,7 @@
       fSize( 10 ), fMinSize( HTML_DEFAULT_MIN_FONT_SIZE )
 
 {
+  KGlobal::locale()->insertCatalog("kcmcss");
   QVBoxLayout* l=new QVBoxLayout(this);
   KTabWidget* tab=new KTabWidget(this);
   l->addWidget(tab);
Index: apps/konqueror/settings/konqhtml/khtml_general.desktop
===================================================================
--- apps/konqueror/settings/konqhtml/khtml_general.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/konqueror/settings/konqhtml/khtml_general.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -54,7 +54,7 @@
 Name[ml]=സാധാരണ
 Name[mr]=सामान्य
 Name[nb]=Generelt
-Name[nds]=Allgemeen
+Name[nds]=Allmeen
 Name[ne]=सामन्य
 Name[nl]=Algemeen
 Name[nn]=Generelt
@@ -127,7 +127,7 @@
 Comment[ml]=കോണ്‍ക്വററിന്റെ പൊതുവായ പെരുമാറ്റം ക്രമീകരിയ്ക്കുക
 Comment[mr]=सामान्य कॉन्करर वर्तन संयोजीत करा
 Comment[nb]=Tilpass Konquerors generelle oppførsel
-Comment[nds]=Dat allgemeen Bedregen vun Konqueror instellen
+Comment[nds]=Allmeen dat Bedregen vun Konqueror instellen
 Comment[nl]=Algemeen gedrag van Konqueror instellen
 Comment[nn]=Sett opp generell Konqueror-åtferd
 Comment[or]=ସାଧାରଣ Konqueror ଆଚରଣକୁ ବିନ୍ୟାସକରନ୍ତ
Index: apps/konqueror/settings/konqhtml/khtml_filter.desktop
===================================================================
--- apps/konqueror/settings/konqhtml/khtml_filter.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/konqueror/settings/konqhtml/khtml_filter.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -138,7 +138,7 @@
 Comment[mr]=कॉन्करर एडब्लॉक फिल्टर संयोजीत करा
 Comment[ms]=Konfigur penapis AdBlocK Konqueror
 Comment[nb]=Tilpass Konquerors reklamefiltre
-Comment[nds]=Hier kannst Du Konquerors Warvenfilter instellen
+Comment[nds]=Konqueror sien Warvenfilter instellen
 Comment[ne]=कन्क्वेरर AdBlocK फिल्टर कन्फिगर गर्नुहोस्
 Comment[nl]=Filters voor blokkeren van advertenties in Konqueror instellen
 Comment[nn]=Tilpass reklamefiltera til Konqueror
Index: apps/konqueror/settings/konqhtml/khtml_java_js.desktop
===================================================================
--- apps/konqueror/settings/konqhtml/khtml_java_js.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/konqueror/settings/konqhtml/khtml_java_js.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -127,7 +127,7 @@
 Comment[mr]=जावा व जावास्क्रिप्ट वर्तन संयोजीत करा
 Comment[ms]=Konfigur kelakuan Java dan JavaScript
 Comment[nb]=Tilpasninger Java og JavaScript
-Comment[nds]=Dat Bedregen vun Java un JavaSkript hier instellen
+Comment[nds]=Dat Bedregen vun Java un JavaSkript instellen
 Comment[ne]=जाभा र जाभा स्क्रिप्टकको व्यवहार कन्फिगर गर्नुहोस्
 Comment[nl]=Hier kunt u het gedrag van Java en JavaScript instellen
 Comment[nn]=Innstillingar for Java og JavaScript
Index: apps/konqueror/settings/konqhtml/css/kcmcss.desktop
===================================================================
--- apps/konqueror/settings/konqhtml/css/kcmcss.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/konqueror/settings/konqhtml/css/kcmcss.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -135,7 +135,7 @@
 Comment[mr]=वेब पान प्रस्तुत करण्या करीता उपयोगजोगी स्टाइलशीट संयोजीत करा
 Comment[ms]=Konfigur lembaran gaya yang digunakan untuk merealisasikan laman web
 Comment[nb]=Tilpass stilsett for visning av nettsider
-Comment[nds]=Stell de Stilvörlaag in, de för't Wiesen vun Nettsieden bruukt warrt
+Comment[nds]=De Stilvörlaag instellen, de för't Wiesen vun Nettsieden bruukt warrt
 Comment[ne]=रेन्डर वेब पृष्ठहरू प्रयोग गर्ने शैलीपाना कन्फिगर गर्नुहोस्
 Comment[nl]=Hier kunt u de stijlbladen instellen die gebruikt worden voor de weergave van webpagina's
 Comment[nn]=Oppsett av stilsett for vising av nettsider
Index: apps/konqueror/settings/konqhtml/khtml_appearance.desktop
===================================================================
--- apps/konqueror/settings/konqhtml/khtml_appearance.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/konqueror/settings/konqhtml/khtml_appearance.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -94,87 +94,25 @@
 Name[zh_TW]=外觀
 
 
-Comment=Configure the fonts used on web pages
-Comment[af]=Konfigureer die skriftipes gebruik word op web bladsye
-Comment[ar]=اضبط الخطوط المستخدمة على صفحات الشبكة
-Comment[as]=ৱেব পৃষ্ঠাত ব্যৱহাৰ কৰা আখৰ বিন্যাস কৰক
-Comment[be]=Настаўленні шрыфтоў для выкарыстання на старонках Сеціва
-Comment[be@latin]=Naładź šryfty na sieciŭnym staronkach
-Comment[bg]=Настройване на шрифтовете за уеб страниците
-Comment[bn]=ওয়েব পেজ-এ ব্যবহৃত ফন্ট কনফিগার করুন
-Comment[bn_IN]=ওয়েব পেজে ব্যবহৃত ফন্ট কনফিগার করুন
-Comment[br]=Kefluniañ an nodrezhoù evit ar pajennoù gwiad
-Comment[ca]=Configura els tipus de lletra emprats a les pàgines web
-Comment[cs]=Nastavení písem pro webové stránky
-Comment[csb]=Kònfigùracëjô fòntów na internetowëch starnach
-Comment[cy]=Ffurfweddu'r ffontiau sy'n cael eu defnyddio ar dudalennau gwe
-Comment[da]=Indstil de skrifttyper der bruges på netsider
-Comment[de]=Schriften für die Darstellung von Webseiten festlegen
-Comment[el]=Ρυθμίστε τις γραμματοσειρές που χρησιμοποιούνται στις ιστοσελίδες
-Comment[eo]=Agordi la tiparojn kiuj estas uzataj sur TTT-paĝoj
-Comment[es]=Configuración de los tipos de letra usados en páginas web
-Comment[et]=Veebilehtedel kasutatavate fontide seadistamine
-Comment[eu]=Konfiguratu web orrietan erabiliko diren letra-tipoak
-Comment[fa]=پیکربندی قلمهای استفاده‌شده در صفحات وب
-Comment[fi]=Verkkosivujen kirjasimet
-Comment[fr]=Configuration des polices utilisées sur les pages Web
-Comment[fy]=Hjir kinne jo de lettertypen die brûkt wurde op de websiden ynstelle
-Comment[ga]=Cumraigh na clónna a úsáidtear ar leathanaigh Ghréasáin
-Comment[gl]=Configura os tipos de letra empregadas nas páxinas web
-Comment[gu]=વેબ પાનાંઓ પર વાપરવાનાં ફોન્ટ રૂપરેખાંકિત કરો
-Comment[he]=שינוי הגדרות הגופנים שנעשה בהם שימוש בדפי אינטרנט
-Comment[hi]=वेब पृष्ठ पर उपयोग में आने वाले फ़ॉन्ट्स कॉन्फ़िगर करें
-Comment[hr]=Konfiguriranje fontova upotrijebljenih na web-stranicama
-Comment[hsb]=Pisma na web-stronach nastajić
-Comment[hu]=A Konqueror webböngésző betűtípusainak beállítása
+Comment=Configure how to display web pages
+Comment[ca]=Configura com visualitzar les pàgines web
+Comment[da]=Indstil hvordan netsider skal vises
+Comment[de]=Einstellungen für die Darstellung von Webseiten
+Comment[et]=Veebilehtede väljanägemise seadistamine
+Comment[eu]=Konfiguratu web orriak nola bistaratu
+Comment[gl]=Configura como se mostran as páxinas web
 Comment[is]=Stillta letur sem notað er á vefsíðum
-Comment[it]=Configura i caratteri da usare nelle pagine web
-Comment[ja]=ウェブページで使用するフォントを設定
-Comment[ka]=ვებ გვერდების შრიფტების კონფიგურაცია
-Comment[kk]=Веб парақтарда қолданатын қаріптер
-Comment[km]=កំណត់​រចនាសម្ព័ន្ធ​ពុម្ព​អក្សរ​ដែល​បាន​​ប្រើ​លើ​ទំព័រ​បណ្ដាញ
-Comment[kn]=ಜಾಲ ಪುಟಗಳಲ್ಲಿ ಬಳಸಲಾಗುವ ಅಕ್ಷರಶೈಲಿಗಳನ್ನು ಸಂರಚಿಸು
-Comment[ko]=웹 페이지의 글꼴 설정
-Comment[ku]=Curenivîsên malperan ava bike
-Comment[lt]=Čia galite konfigūruoti šriftus, naudojamus tinklalapiuose
-Comment[lv]=Šeit jūs varat konfigurēt tīmekļa lapās lietotos fontus
-Comment[mai]=वेब पृष्ठ पर उपयोग मे आबैबला फान्ट्स बिन्यस्त करू
-Comment[mk]=Конфигурирајте ги фонтовите што се користат за веб-страници
-Comment[ml]=വെബ്താളുകളിലുപയോഗിയ്ക്കുന്ന അക്ഷരസഞ്ചയങ്ങള്‍ ക്രമീകരിയ്ക്കുക
-Comment[mr]=वेब पानांवरील वापरले गेलेले फॉन्ट्स संयोजीत करा
-Comment[ms]=Konfigur fon yang digunakan dalam halaman web
-Comment[nb]=Tilpass hvilke skrifttyper som brukes på nettsider
-Comment[nds]=De Schriftoorden för Nettsieden instellen
-Comment[ne]=वेब पृष्ठहरूमा प्रयोग गरिने फन्ट कन्फिगर गर्नुहोस्
-Comment[nl]=Hier kunt u de lettertypen die op webpagina's gebruikt worden instellen
-Comment[nn]=Oppsett av skrifter for nettsider
-Comment[or]=ୱେବପୃଷ୍ଠାଗୁଡ଼ିକରେ ବ୍ୟବହୃତ ହେଉଥିବା ଅକ୍ଷରରୂପଗୁଡ଼ିକୁ ବିନ୍ୟାସ କରନ୍ତୁ
-Comment[pa]=ਵੈੱਬ ਸਫੇ 'ਤੇ ਫੋਂਟ ਸੰਰਚਨਾ
-Comment[pl]=Konfiguracja czcionek na stronach internetowych
-Comment[pt]=Configurar os tipos de letra usados nas páginas Web
-Comment[pt_BR]=Configurar as fontes usadas nas páginas da Internet
-Comment[ro]=Configurează fonturile utilizate în paginile web
-Comment[ru]=Настройка шрифтов веб-страниц
-Comment[se]=Heivet makkár fonttat geavahuvvojit fierpmádatsiidduin
-Comment[sk]=Nastavenie písiem pre webové stránky
-Comment[sl]=Nastavitve pisav, prikazanih na spletnih straneh
-Comment[sr]=Подешавање фонтова који се користе на веб страницама
-Comment[sr@latin]=Podešavanje fontova koji se koriste na veb stranicama
-Comment[sv]=Anpassa teckensnitt som används på webbsidor
-Comment[ta]=வலைபக்கங்களில் பயன்படுத்தப்படும் எழுத்துருக்களை வடிவமை
-Comment[te]=వెబ్ పుటలపై ఉపయోగించే ఫాంట్‍స్‌ను ఆకృతీకరించుము
-Comment[tg]=Танзимоти ҳарфҳои саҳифаҳои Интернетӣ
-Comment[th]=ปรับแต่งแบบอักษรที่จะใช้กับหน้าเว็บ
-Comment[tr]=Web sayfalarında kullanılan yazı tiplerini yapılandır
-Comment[uk]=Налаштування шрифтів для сторінок Тенет
-Comment[uz]=Veb-sahifalarida ishlatiladigan shriftlarni moslash
-Comment[uz@cyrillic]=Веб-саҳифаларида ишлатиладиган шрифтларни мослаш
-Comment[vi]=Cấu hình phông chữ sử dụng trên các trang mạng
-Comment[wa]=Apontyî les fontes eployeyes ezès pådjes waibe
-Comment[x-test]=xxConfigure the fonts used on web pagesxx
-Comment[xh]=Qwlasela uhlobo lwamagama asebenziswe kumaphepha e web
-Comment[zh_CN]=配置网页所使用的字体
-Comment[zh_TW]=設定使用於網頁的字型
+Comment[nds]=Instellen, wodennig Nettsieden wiest warrt
+Comment[nl]=Hier kunt u instellen hoe webpagina's te tonen
+Comment[pl]=Konfiguracja sposobu wyświetlania stron WWW
+Comment[pt]=Configurar a apresentação das páginas Web
+Comment[pt_BR]=Configurar como exibir as páginas da Web
+Comment[sl]=Nastavitve načina prikazovanja spletnih strani
+Comment[sv]=Anpassa hur webbsidor ska visas
+Comment[uk]=Налаштування способу показу веб-сторінок
+Comment[x-test]=xxConfigure how to display web pagesxx
+Comment[zh_CN]=配置如何显示网页
+Comment[zh_TW]=設定如何顯示網頁
 X-KDE-Keywords=konqueror,kfm,browser,html,web,www,fonts,charsets,character sets,encoding
 X-KDE-Keywords[be]=Вандроўнік,Сеціва,Шрыфт,Знаказборы,Знаказбор,konqueror,kfm,browser,html,web,www,fonts,charsets,character sets,encoding
 X-KDE-Keywords[bg]=Keywords=уеб, шрифтове, страници, браузър, Интернет, konqueror, kfm, browser, html, web, www, fonts, charsets, character sets, encoding
Index: apps/konqueror/sidebar/konq_sidebartng.desktop
===================================================================
--- apps/konqueror/sidebar/konq_sidebartng.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/konqueror/sidebar/konq_sidebartng.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -51,7 +51,7 @@
 Name[mr]=संचारन पटल
 Name[ms]=Panel Navigasi
 Name[nb]=Navigasjonspanel
-Name[nds]=Navigatschoonspaneel
+Name[nds]=Navigeer-Paneel
 Name[ne]=नेभिगेसन प्यानल
 Name[nl]=Navigatiepaneel
 Name[nn]=Navigasjonspanel
@@ -73,7 +73,7 @@
 Name[te]=నావిగేషన్ ప్యానల్
 Name[tg]=Панели намоишгар
 Name[th]=แถบนำทาง
-Name[tr]=Dolaşma Paneli
+Name[tr]=Gezgin Paneli
 Name[uk]=Панель навігації
 Name[uz]=Yoʻlchi paneli
 Name[uz@cyrillic]=Йўлчи панели
Index: apps/konqueror/shellcmdplugin/kshellcmdplugin.desktop
===================================================================
--- apps/konqueror/shellcmdplugin/kshellcmdplugin.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/konqueror/shellcmdplugin/kshellcmdplugin.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -59,7 +59,7 @@
 Name[mr]=शेल आदेश प्लगइन
 Name[ms]=Plugin Arahan Shell
 Name[nb]=Programtillegg for skallkommandoer
-Name[nds]=Plugin för Konsoolbefehlen
+Name[nds]=Moduul för Konsoolbefehlen
 Name[ne]=शेल आदेश प्लगइन
 Name[nl]=Schellcommando-plugin
 Name[nn]=Programtillegg for skalkommandoar
@@ -135,7 +135,7 @@
 Comment[mr]=कॉन्करर करीता शेल आदेश प्लगइन
 Comment[ms]=Plugin Arahan Shell untuk Konqueror
 Comment[nb]=Programtillegg for skallkommandoer i Konqueror
-Comment[nds]=Konsoolbefehl-Plugin för Konqueror
+Comment[nds]=Konsoolbefehl-Moduul för Konqueror
 Comment[ne]=कन्क्वेररका लागि शेल आदेश प्लगइन
 Comment[nl]=Een shellcommando-plugin voor Konqueror
 Comment[nn]=Programtillegg for skalkommandoar i Konqueror
Index: apps/dolphin/src/treeviewsidebarpage.h
===================================================================
--- apps/dolphin/src/treeviewsidebarpage.h	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/dolphin/src/treeviewsidebarpage.h	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -80,6 +80,9 @@
 
     /** @see QWidget::contextMenuEvent() */
     virtual void contextMenuEvent(QContextMenuEvent* event);
+    
+    /** @see QWidget::keyPressEvent() */
+    virtual void keyPressEvent(QKeyEvent* event);
 
 private slots:
     /**
Index: apps/dolphin/src/dolphinmainwindow.cpp
===================================================================
--- apps/dolphin/src/dolphinmainwindow.cpp	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/dolphin/src/dolphinmainwindow.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -1301,6 +1301,10 @@
         name = url.fileName();
         if (name.isEmpty()) {
             name = url.protocol();
+        } else {
+            // Make sure that a '&' inside the directory name is displayed correctly
+            // and not misinterpreted as a keyboard shortcut in QTabBar::setTabText()
+            name.replace('&', "&&");
         }
     }
     return name;
Index: apps/dolphin/src/dolphindetailsview.cpp
===================================================================
--- apps/dolphin/src/dolphindetailsview.cpp	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/dolphin/src/dolphindetailsview.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -900,7 +900,7 @@
         // reasons the exact necessary width for full visible names is
         // only checked for up to 200 items:
         const int rowCount = model()->rowCount();
-        if (rowCount < 200) {
+        if (rowCount > 0 && rowCount < 200) {
             const int nameWidth = sizeHintForColumn(DolphinModel::Name);
             if (nameWidth + requiredWidth <= viewport()->width()) {
                 columnWidth[KDirModel::Name] = viewport()->width() - requiredWidth;
Index: apps/dolphin/src/treeviewsidebarpage.cpp
===================================================================
--- apps/dolphin/src/treeviewsidebarpage.cpp	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/dolphin/src/treeviewsidebarpage.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -159,7 +159,7 @@
 
         new FolderExpander(m_treeView, m_proxyModel);
 
-        connect(m_treeView, SIGNAL(activated(const QModelIndex&)),
+        connect(m_treeView, SIGNAL(clicked(const QModelIndex&)),
                 this, SLOT(updateActiveView(const QModelIndex&)));
         connect(m_treeView, SIGNAL(urlsDropped(const QModelIndex&, QDropEvent*)),
                 this, SLOT(dropUrls(const QModelIndex&, QDropEvent*)));
@@ -191,6 +191,17 @@
     contextMenu.open();
 }
 
+void TreeViewSidebarPage::keyPressEvent(QKeyEvent* event)
+{
+    const int key = event->key();
+    if ((key == Qt::Key_Enter) || (key == Qt::Key_Return)) {
+        event->accept();
+        updateActiveView(m_treeView->currentIndex());
+    } else {
+        SidebarPage::keyPressEvent(event);
+    }
+}
+
 void TreeViewSidebarPage::updateActiveView(const QModelIndex& index)
 {
     const QModelIndex dirIndex = m_proxyModel->mapToSource(index);
Index: apps/dolphin/src/folderexpander.cpp
===================================================================
--- apps/dolphin/src/folderexpander.cpp	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/dolphin/src/folderexpander.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -112,7 +112,9 @@
     Q_ASSERT(m_dirModel != 0);
     KFileItem itemToExpand = m_dirModel->itemForIndex(indexToExpand);
 
-    if (itemToExpand.isNull()) {
+    if (itemToExpand.isNull() || itemToExpand == m_dirModel->itemForIndex(QModelIndex())) {
+        // The second clause occurs when we are expanding the folder represented
+        // by the view, which is a case we should ignore (#182618).
         return;
     }
 
Index: apps/doc/kdepasswd/index.docbook
===================================================================
--- apps/doc/kdepasswd/index.docbook	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/doc/kdepasswd/index.docbook	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -4,7 +4,7 @@
   <!ENTITY % addindex "IGNORE">
   <!ENTITY % English "INCLUDE">
 ]>
-<article>
+<article lang="&language;">
 <articleinfo>
 <authorgroup>
 <author>
@@ -145,4 +145,4 @@
 
 vim:tabstop=2:shiftwidth=2:expandtab
 kate: space-indent on; indent-width 2; tab-width 2; indent-mode none;
--->
\ No newline at end of file
+-->
Index: apps/lib/konq/konq_operations.cpp
===================================================================
--- apps/lib/konq/konq_operations.cpp	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/lib/konq/konq_operations.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -287,7 +287,10 @@
         // Ok, now we need destItem.
         if ( !destItem.isNull() )
         {
-            op->asyncDrop( destItem ); // we have it already
+            // We have it already, we could just call asyncDrop.
+            // But popping up a menu in the middle of a DND operation confuses and crashes Qt (#157630)
+            // So let's delay it.
+            QMetaObject::invokeMethod(op, "asyncDrop", Q_ARG(KFileItem, destItem));
         }
         else
         {
Index: apps/lib/konq/Templates/linkProgram.desktop
===================================================================
--- apps/lib/konq/Templates/linkProgram.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/lib/konq/Templates/linkProgram.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -8,9 +8,11 @@
 Name[bg]=Връзка към програма...
 Name[bn]=অ্যাপলিকেশন-এ লিঙ্ক...
 Name[bn_IN]=অ্যাপ্লিকেশন নির্দেশকারী লিঙ্ক...
+Name[br]=Liamm ouzh un arload ...
 Name[ca]=Enllaç a l'aplicació...
 Name[cs]=Odkaz na aplikaci...
 Name[csb]=Lënk do programë...
+Name[cy]=Cyswllt i Gymhwysiad...
 Name[da]=Link til program...
 Name[de]=Verknüpfung zu Programm ...
 Name[el]=Δεσμός με εφαρμογή...
@@ -32,6 +34,7 @@
 Name[is]=Tengi við forrit...
 Name[it]=Collegamento ad applicazione...
 Name[ja]=アプリケーションへのリンク...
+Name[ka]=პროგრამის ბმული...
 Name[kk]=Қолданбаға сілтеме...
 Name[km]=តភ្ជាប់​ទៅ​កម្មវិធី...
 Name[kn]=ಅನ್ವಯಕ್ಕೆ ಕೊಂಡಿ...
@@ -42,6 +45,7 @@
 Name[mk]=Врска до апликација...
 Name[ml]=പ്രയോഗത്തിലേക്കുള്ള കണ്ണി...
 Name[mr]=अनुप्रयोग करीता लिंक...
+Name[ms]=Paut ke Aplikasi...
 Name[nb]=Lenke til program …
 Name[nds]=Link na Programm...
 Name[nl]=Koppeling naar programma...
@@ -67,6 +71,7 @@
 Name[uk]=Посилання на програму...
 Name[uz]=Dastur bilan yangi bogʻ...
 Name[uz@cyrillic]=Дастур билан янги боғ...
+Name[vi]=Liên kết tới ứng dụng...
 Name[wa]=Loyî å programe...
 Name[x-test]=xxLink to Application...xx
 Name[zh_CN]=链接到应用程序...
Index: apps/lib/konq/Templates/linkMO.desktop
===================================================================
--- apps/lib/konq/Templates/linkMO.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/lib/konq/Templates/linkMO.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -85,9 +85,11 @@
 Comment[bg]=Ново MO устройство
 Comment[bn]=নতুন MO ডিভাইস
 Comment[bn_IN]=নতুন MO ডিভাইস
+Comment[br]=Trobarzhell MO nevez
 Comment[ca]=Dispositiu MO nou
 Comment[cs]=Nové MO zařízení
 Comment[csb]=Nowi nëk MO
+Comment[cy]=Dyfais MO Newydd
 Comment[da]=Ny MO-enhed
 Comment[de]=Neues MO-Laufwerk
 Comment[el]=Νέα συσκευή MO
@@ -109,6 +111,7 @@
 Comment[is]=Nýtt MO-drif
 Comment[it]=Nuovo dispositivo MO
 Comment[ja]=新しい MO デバイス
+Comment[ka]=ახალი MO მოწყობილობა
 Comment[kk]=Жаңа MO
 Comment[km]=ឧបករណ៍ MO ថ្មី
 Comment[kn]=ಹೊಸ MO ಸಾಧನ
@@ -119,6 +122,7 @@
 Comment[mk]=Нов MO уред
 Comment[ml]=പുതിയ MO ഉപകരണം...
 Comment[mr]=नविन MO साधन
+Comment[ms]=Peranti MO Baru
 Comment[nb]=Ny MO-enhet
 Comment[nds]=Niege MO-Reedschap
 Comment[nl]=Nieuw MO-apparaat
@@ -142,6 +146,9 @@
 Comment[th]=อุปกรณ์ MO ตัวใหม่
 Comment[tr]=Yeni MO Aygıtı
 Comment[uk]=Новий пристрій MO
+Comment[uz]=Yangi MO uskunasi
+Comment[uz@cyrillic]=Янги MO ускунаси
+Comment[vi]=Thiết bị MO mới
 Comment[wa]=Novea éndjin MO
 Comment[x-test]=xxNew MO Devicexx
 Comment[zh_CN]=新建 MO 设备
Index: apps/konsole/desktop/konsole.desktop
===================================================================
--- apps/konsole/desktop/konsole.desktop	(.../tags/KDE/4.2.0/kdebase)	(wersja 920546)
+++ apps/konsole/desktop/konsole.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 920546)
@@ -92,5 +92,6 @@
 GenericName[zh_CN]=终端
 GenericName[zh_TW]=終端機
 X-DBUS-StartupType=Unique
+X-DBUS-ServiceName=org.kde.konsole
 X-KDE-AuthorizeAction=shell_access
 Categories=Qt;KDE;System;TerminalEmulator;
Index: runtime/cmake/modules/FindPulseAudio.cmake
===================================================================
Index: runtime/kdedglobalaccel/kdedglobalaccel.desktop
===================================================================
Index: runtime/knotify/kde.notifyrc
===================================================================
Index: runtime/phonon/platform_kde/phonon.notifyrc
===================================================================
Index: runtime/phonon/kcm/backendselection.cpp
===================================================================
Index: runtime/menu/desktop/kde-games-logic.directory
===================================================================
Index: runtime/doc/kcontrol/emoticons/index.docbook
===================================================================
Index: runtime/doc/kcontrol/kcmcss/index.docbook
===================================================================
Index: runtime/kwalletd/kbetterthankdialog.cpp
===================================================================
Index: runtime/kwalletd/kwalletd.cpp
===================================================================
Index: runtime/kwalletd/main.cpp
===================================================================
Index: runtime/kwalletd/kbetterthankdialogbase.ui
===================================================================
Index: runtime/kioslave/trash/kcmtrash.desktop
===================================================================
Index: runtime/kioslave/trash/trashimpl.cpp
===================================================================
Index: runtime/kioslave/trash/discspaceutil.cpp
===================================================================
Index: runtime/kioslave/desktop/directory.trash
===================================================================
Index: runtime/nepomuk/services/strigi/nepomukstrigiservice.desktop
===================================================================
Index: runtime/nepomuk/services/strigi/systray.cpp
===================================================================
Index: runtime/nepomuk/services/strigi/statuswidget.cpp
===================================================================
Index: runtime/nepomuk/services/strigi/nepomukstrigiservice.notifyrc
===================================================================
Index: runtime/nepomuk/services/storage/nepomukstorage.notifyrc
===================================================================
Index: runtime/kstyles/oxygen/oxygen.cpp
===================================================================
Index: runtime/drkonqi/presets/developerrc
===================================================================
Index: runtime/drkonqi/presets/enduserrc
===================================================================
Index: runtime/l10n/eastafrica.desktop
===================================================================
Index: runtime/l10n/ec/entry.desktop
===================================================================
Index: workspace/ksmserver/shutdowndlg.cpp
===================================================================
Index: workspace/kcontrol/kfontinst/lib/FcEngine.cpp
===================================================================
Index: workspace/kcontrol/screensaver/scrnsave.cpp
===================================================================
Index: workspace/kcontrol/autostart/autostart.desktop
===================================================================
Index: workspace/kcontrol/randr/outputgraphicsitem.cpp
===================================================================
Index: workspace/kcontrol/randr/outputconfig.cpp
===================================================================
Index: workspace/kcontrol/randr/outputgraphicsitem.h
===================================================================
Index: workspace/kcontrol/randr/krandrtray.desktop
===================================================================
Index: workspace/kcontrol/kdm/background/patterns/triangles.desktop
===================================================================
Index: workspace/kcontrol/kdm/background/patterns/fish.desktop
===================================================================
Index: workspace/doc/kcontrol/powerdevil/index.docbook
===================================================================
Index: workspace/doc/kcontrol/autostart/index.docbook
===================================================================
Index: workspace/doc/kcontrol/splashscreen/index.docbook
===================================================================
Index: workspace/doc/kcontrol/kcmaccess/index.docbook
===================================================================
Index: workspace/doc/kcontrol/joystick/index.docbook
===================================================================
Index: workspace/doc/kcontrol/kcmdisplay/index.docbook
===================================================================
Index: workspace/systemsettings/categories/settings-general.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-advanced-user-settings.desktop
===================================================================
Index: workspace/kwrited/kwrited.notifyrc
===================================================================
Index: workspace/wallpapers/HighTide/metadata.desktop
===================================================================
Index: workspace/powerdevil/kcmodule/powerdevilconfig.desktop
===================================================================
Index: workspace/powerdevil/powerdevil.notifyrc
===================================================================
Index: workspace/krunner/screensaver/saverengine.cpp
===================================================================
Index: workspace/krunner/screensaver/xautolock.cpp
===================================================================
Index: workspace/krunner/interfaces/quicksand/qs_matchview.h
===================================================================
Index: workspace/krunner/interfaces/quicksand/qs_dialog.cpp
===================================================================
Index: workspace/krunner/interfaces/quicksand/qs_dialog.h
===================================================================
Index: workspace/krunner/interfaces/quicksand/qs_matchview.cpp
===================================================================
Index: workspace/krunner/interfaces/default/interface.cpp
===================================================================
Index: workspace/krunner/interfaces/default/resultscene.h
===================================================================
Index: workspace/krunner/interfaces/default/resultscene.cpp
===================================================================
Index: workspace/krunner/lock/lockprocess.cc
===================================================================
Index: workspace/libs/kephal/xrandr12/randroutput.cpp
===================================================================
Index: workspace/libs/kephal/kded_kephal/kded_kephal.desktop
===================================================================
Index: workspace/libs/ksysguard/processcore/processes.h
===================================================================
Index: workspace/libs/ksysguard/processui/ProcessModel.h
===================================================================
Index: workspace/libs/ksysguard/processui/ksysguardprocesslist.h
===================================================================
Index: workspace/libs/ksysguard/processui/ProcessFilter.h
===================================================================
Index: workspace/khotkeys/app/khotkeys.desktop
===================================================================
Index: workspace/khotkeys/data/kde32b1.khotkeys
===================================================================
Index: workspace/kwin/effects/videorecord.cpp
===================================================================
Index: workspace/kwin/effects/videorecord_config.cpp
===================================================================
Index: workspace/kwin/effects/videorecord_config.desktop
===================================================================
Index: workspace/kwin/effects/videorecord_config.h
===================================================================
Index: workspace/kwin/effects/videorecord.h
===================================================================
Index: workspace/kwin/effects/videorecord.desktop
===================================================================
Index: workspace/kwin/effects/invert.desktop
===================================================================
Index: workspace/kwin/effects/taskbarthumbnail.desktop
===================================================================
Index: workspace/kwin/effects/shadow.cpp
===================================================================
Index: workspace/kwin/effects/maketransparent.desktop
===================================================================
Index: workspace/kwin/effects/sharpen.desktop
===================================================================
Index: workspace/kwin/effects/blur.desktop
===================================================================
Index: workspace/kwin/effects/fallapart.desktop
===================================================================
Index: workspace/kwin/effects/explosion.desktop
===================================================================
Index: workspace/kwin/effects/shadow_config.cpp
===================================================================
Index: workspace/kwin/effects/zoom.desktop
===================================================================
Index: workspace/kwin/effects/thumbnailaside.desktop
===================================================================
Index: workspace/kwin/effects/snow.cpp
===================================================================
Index: workspace/kwin/effects/magnifier.desktop
===================================================================
Index: workspace/kwin/effects/cube.desktop
===================================================================
Index: workspace/kwin/effects/snow.desktop
===================================================================
Index: workspace/kwin/effects/magiclamp.desktop
===================================================================
Index: workspace/kwin/effects/dimscreen.desktop
===================================================================
Index: workspace/kwin/effects/scalein.desktop
===================================================================
Index: workspace/kwin/effects/diminactive_config.desktop
===================================================================
Index: workspace/kwin/effects/minimizeanimation.desktop
===================================================================
Index: workspace/kwin/effects/shadow.h
===================================================================
Index: workspace/kwin/effects/mousemark.desktop
===================================================================
Index: workspace/kwin/effects/dialogparent.desktop
===================================================================
Index: workspace/kwin/effects/trackmouse.desktop
===================================================================
Index: workspace/kwin/effects/desktopgrid.desktop
===================================================================
Index: workspace/kwin/effects/fade.desktop
===================================================================
Index: workspace/kwin/effects/shadow.desktop
===================================================================
Index: workspace/kwin/effects/diminactive.desktop
===================================================================
Index: workspace/kwin/effects/slide.desktop
===================================================================
Index: workspace/kwin/effects/data/snow.vert
===================================================================
Index: workspace/kwin/effects/CMakeLists.txt
===================================================================
Index: workspace/kwin/effects/test/demo_showpicture.desktop
===================================================================
Index: workspace/kwin/effects/test/videorecord.h
===================================================================

Zmiany atrybutów dla: workspace/kwin/effects/test/videorecord.h
___________________________________________________________________
Dodane: svn:mergeinfo
   Połączono zmiany /branches/KDE/4.0/kdebase/workspace/kwin/effects/videorecord.h:r758600,758610

Index: workspace/kwin/effects/test/videorecord_config.desktop
===================================================================

Zmiany atrybutów dla: workspace/kwin/effects/test/videorecord_config.desktop
___________________________________________________________________
Dodane: svn:mergeinfo
   Połączono zmiany /branches/KDE/4.0/kdebase/workspace/kwin/effects/videorecord_config.desktop:r758600,758610

Index: workspace/kwin/effects/test/demo_liquid.desktop
===================================================================
Index: workspace/kwin/effects/test/demo_shiftworkspaceup.desktop
===================================================================
Index: workspace/kwin/effects/test/CMakeLists.txt
===================================================================
Index: workspace/kwin/effects/test/drunken.desktop
===================================================================
Index: workspace/kwin/effects/test/videorecord_config.h
===================================================================

Zmiany atrybutów dla: workspace/kwin/effects/test/videorecord_config.h
___________________________________________________________________
Dodane: svn:mergeinfo
   Połączono zmiany /branches/KDE/4.0/kdebase/workspace/kwin/effects/videorecord_config.h:r758600,758610

Index: workspace/kwin/effects/test/videorecord.cpp
===================================================================

Zmiany atrybutów dla: workspace/kwin/effects/test/videorecord.cpp
___________________________________________________________________
Dodane: svn:mergeinfo
   Połączono zmiany /branches/KDE/4.0/kdebase/workspace/kwin/effects/videorecord.cpp:r758600,758610

Index: workspace/kwin/effects/test/videorecord_config.cpp
===================================================================

Zmiany atrybutów dla: workspace/kwin/effects/test/videorecord_config.cpp
___________________________________________________________________
Dodane: svn:mergeinfo
   Połączono zmiany /branches/KDE/4.0/kdebase/workspace/kwin/effects/videorecord_config.cpp:r758600,758610

Index: workspace/kwin/effects/test/videorecord.desktop
===================================================================

Zmiany atrybutów dla: workspace/kwin/effects/test/videorecord.desktop
___________________________________________________________________
Dodane: svn:mergeinfo
   Połączono zmiany /branches/KDE/4.0/kdebase/workspace/kwin/effects/videorecord.desktop:r758600,758610

Index: workspace/kwin/effects/sphere.desktop
===================================================================
Index: workspace/kwin/useractions.cpp
===================================================================
Index: workspace/kwin/kwin.notifyrc
===================================================================
Index: workspace/kdm/config.def
===================================================================
Index: workspace/kdm/backend/resource.c
===================================================================
Index: workspace/plasma/dataengines/applicationjobs/plasma-dataengine-applicationjobs.desktop
===================================================================
Index: workspace/plasma/dataengines/executable/plasma-dataengine-executable.desktop
===================================================================
Index: workspace/plasma/dataengines/dict/plasma-dataengine-dict.desktop
===================================================================
Index: workspace/plasma/dataengines/filebrowser/plasma-dataengine-filebrowser.desktop
===================================================================
Index: workspace/plasma/dataengines/nowplaying/plasma-dataengine-nowplaying.desktop
===================================================================
Index: workspace/plasma/dataengines/network/plasma-dataengine-network.desktop
===================================================================
Index: workspace/plasma/dataengines/powermanagement/plasma-dataengine-powermanagement.desktop
===================================================================
Index: workspace/plasma/dataengines/soliddevice/plasma-dataengine-soliddevice.desktop
===================================================================
Index: workspace/plasma/dataengines/systemmonitor/plasma-dataengine-systemmonitor.desktop
===================================================================
Index: workspace/plasma/dataengines/time/plasma-dataengine-time.desktop
===================================================================
Index: workspace/plasma/dataengines/places/plasma-dataengine-places.desktop
===================================================================
Index: workspace/plasma/dataengines/notifications/plasma-dataengine-notifications.desktop
===================================================================
Index: workspace/plasma/shells/desktop/desktopview.cpp
===================================================================
Index: workspace/plasma/shells/desktop/plasmaapp.cpp
===================================================================
Index: workspace/plasma/shells/desktop/plasmaapp.h
===================================================================
Index: workspace/plasma/runners/places/plasma-runner-places.desktop
===================================================================
Index: workspace/plasma/applets/webbrowser/webbrowser.cpp
===================================================================
Index: workspace/plasma/applets/webbrowser/plasma-applet-webbrowser.desktop
===================================================================
Index: workspace/plasma/applets/systemtray/core/job.cpp
===================================================================
Index: workspace/plasma/applets/icon/plasma-applet-icon.desktop
===================================================================
Index: workspace/plasma/applets/devicenotifier/plasma-applet-devicenotifier.desktop
===================================================================
Index: workspace/plasma/applets/quicklaunch/plasma-applet-quicklaunch.desktop
===================================================================
Index: workspace/plasma/applets/calendar/plasma-applet-calendar.desktop
===================================================================
Index: workspace/plasma/applets/system-monitor/plasma-applet-sm_cpu.desktop
===================================================================
Index: workspace/plasma/applets/system-monitor/plasma-applet-sm_hwinfo.desktop
===================================================================
Index: workspace/plasma/applets/system-monitor/plasma-applet-sm_temperature.desktop
===================================================================
Index: workspace/plasma/applets/system-monitor/plasma-applet-sm_net.desktop
===================================================================
Index: workspace/plasma/applets/system-monitor/plasma-applet-sm_hdd.desktop
===================================================================
Index: workspace/plasma/applets/kickoff/simpleapplet/simpleapplet.cpp
===================================================================
Index: workspace/plasma/applets/kickoff/simpleapplet/simpleapplet.h
===================================================================
Index: workspace/plasma/applets/kickoff/core/applicationmodel.cpp
===================================================================
Index: workspace/plasma/scriptengines/python/pydataengine.py
===================================================================
Index: workspace/plasma/scriptengines/python/plasma_importer.py
===================================================================
Index: workspace/plasma/scriptengines/python/examples/dataengines/pytime/metadata.desktop
===================================================================
Index: workspace/plasma/scriptengines/python/examples/applets/pyclock/metadata.desktop
===================================================================
Index: workspace/plasma/scriptengines/ruby/examples/dataengines/dbpedia_albums/metadata.desktop
===================================================================
Index: workspace/plasma/scriptengines/ruby/examples/dataengines/time/metadata.desktop
===================================================================
Index: workspace/plasma/scriptengines/ruby/examples/applets/webapplet/metadata.desktop
===================================================================
Index: workspace/plasma/scriptengines/ruby/examples/applets/clock/metadata.desktop
===================================================================
Index: workspace/plasma/scriptengines/webkit/plasma-packagestructure-web.desktop
===================================================================
Index: workspace/plasma/scriptengines/javascript/tests/javascript-config-test/metadata.desktop
===================================================================
Index: workspace/plasma/scriptengines/javascript/tests/script-digital-clock/metadata.desktop
===================================================================
Index: workspace/plasma/scriptengines/javascript/tests/script-nowplaying/metadata.desktop
===================================================================
Index: workspace/plasma/scriptengines/javascript/tests/tiger/metadata.desktop
===================================================================
Index: workspace/plasma/scriptengines/qedjescript/plasma-appletscript-qedje.desktop
===================================================================
Index: workspace/plasma/scriptengines/qedjescript/plasma-packagestructure-qedje.desktop
===================================================================
