Index: apps/konqueror/konqueror.desktop
===================================================================
--- apps/konqueror/konqueror.desktop	(.../tags/KDE/4.3.2/kdebase)	(wersja 1032821)
+++ apps/konqueror/konqueror.desktop	(.../branches/KDE/4.3/kdebase)	(wersja 1032821)
@@ -120,7 +120,7 @@
 Comment[he]=מנהל הקבצים ודפדפן האינטרנט של KDE
 Comment[hi]=केडीई फ़ाइल प्रबंधक व वेब ब्राउज़र
 Comment[hne]=केडीई फाइल प्रबंधक अउ वेब ब्राउजर
-Comment[hr]=KDE upravitelj datotekama i web preglednik
+Comment[hr]=KDE-ov upravitelj datoteka i web preglednik
 Comment[hsb]=KDE datajowy manager & web browser
 Comment[hu]=KDE fájlkezelő és webböngésző
 Comment[is]=KDE skráastjóri og vafri
Index: apps/konqueror/konquerorsu.desktop
===================================================================
--- apps/konqueror/konquerorsu.desktop	(.../tags/KDE/4.3.2/kdebase)	(wersja 1032821)
+++ apps/konqueror/konquerorsu.desktop	(.../branches/KDE/4.3/kdebase)	(wersja 1032821)
@@ -38,7 +38,7 @@
 Name[he]=מנהל קבצים - מצב משתמש־על
 Name[hi]=फ़ाइल प्रबंधक- सुपर यूजर मोड
 Name[hne]=फाइल प्रबंधक- सुपर यूजर मोड
-Name[hr]=Upravitelj datotekama – za administratora
+Name[hr]=Upravitelj datoteka – za administratora
 Name[hsb]=Datajowy manager - modus superwužiwarja
 Name[hu]=Fájlkezelő (rendszergazdai mód)
 Name[is]=Skráastjóri (sem kerfisstjóri)
Index: apps/dolphin/src/panels/terminal/terminalpanel.cpp
===================================================================
--- apps/dolphin/src/panels/terminal/terminalpanel.cpp	(.../tags/KDE/4.3.2/kdebase)	(wersja 1032821)
+++ apps/dolphin/src/panels/terminal/terminalpanel.cpp	(.../branches/KDE/4.3/kdebase)	(wersja 1032821)
@@ -23,7 +23,6 @@
 #include <kde_terminal_interface_v2.h>
 #include <kparts/part.h>
 #include <kshell.h>
-#include <kio/netaccess.h>
 
 #include <QBoxLayout>
 #include <QShowEvent>
@@ -58,13 +57,12 @@
     }
 
     Panel::setUrl(url);
-    KUrl mostLocalUrl = KIO::NetAccess::mostLocalUrl(url, 0);
     const bool sendInput = (m_terminal != 0)
                            && (m_terminal->foregroundProcessId() == -1)
                            && isVisible()
-                           && mostLocalUrl.isLocalFile();
+                           && url.isLocalFile();
     if (sendInput) {
-        m_terminal->sendInput("cd " + KShell::quoteArg(mostLocalUrl.toLocalFile()) + '\n');
+        m_terminal->sendInput("cd " + KShell::quoteArg(url.toLocalFile()) + '\n');
     }
 
 }
Index: apps/keditbookmarks/bookmarkmodel.cpp
===================================================================
--- apps/keditbookmarks/bookmarkmodel.cpp	(.../tags/KDE/4.3.2/kdebase)	(wersja 1032821)
+++ apps/keditbookmarks/bookmarkmodel.cpp	(.../branches/KDE/4.3/kdebase)	(wersja 1032821)
@@ -78,7 +78,7 @@
     //Text
     if(index.isValid() && (role == Qt::DisplayRole || role == Qt::EditRole))
     {
-        KBookmark bk = bookmarkForIndex(index);
+        const KBookmark bk = bookmarkForIndex(index);
         if(bk.address().isEmpty())
         {
             if(index.column() == 0)
@@ -100,12 +100,12 @@
                     : subnode.firstChild().toText().data();
             }
             case 3: { //Status column
-                QString text1 = ""; //FIXME favicon state
-                QString text2 = ""; //FIXME link state
-                if(text1.isNull() || text2.isNull())
-                    return QVariant( text1 + text2);
+                QString text1; //FIXME favicon state
+                QString text2; //FIXME link state
+                if(text1.isEmpty() || text2.isEmpty())
+                    return QVariant( text1 + text2 );
                 else
-                    return QVariant( text1 + "  --  " + text2);
+                    return QVariant( text1 + "  --  " + text2 );
             }
             default:
                 return QVariant( QString() ); //can't happen
@@ -141,12 +141,14 @@
                 return Qt::ItemIsEnabled | Qt::ItemIsSelectable | Qt::ItemIsEditable | Qt::ItemIsDragEnabled | Qt::ItemIsDropEnabled;
         }
         else
+        {
             if(index.column() < 3)
-                return Qt::ItemIsEnabled | Qt::ItemIsSelectable | Qt::ItemIsEditable | Qt::ItemIsDragEnabled | Qt::ItemIsDropEnabled;
+                return Qt::ItemIsEnabled | Qt::ItemIsSelectable | Qt::ItemIsEditable | Qt::ItemIsDragEnabled;
+        }
     }
 
     // root
-    return Qt::ItemIsEnabled | Qt::ItemIsSelectable;
+    return Qt::ItemIsEnabled | Qt::ItemIsSelectable | Qt::ItemIsDropEnabled;
 }
 
 bool KBookmarkModel::setData(const QModelIndex &index, const QVariant &value, int role)
@@ -187,7 +189,7 @@
         return createIndex(row, column, d->mRootItem);
 
     TreeItem * item = static_cast<TreeItem *>(parent.internalPointer());
-    if(row == item->childCount()) {// Received drop below last row, simulate drop on last row
+        if(row == item->childCount()) {// Received drop below last row, simulate drop on last row
         return createIndex(row-1, column, item->child(row-1));
     }
 
@@ -284,22 +286,42 @@
 
 bool KBookmarkModel::dropMimeData(const QMimeData * data, Qt::DropAction action, int row, int column, const QModelIndex & parent)
 {
-    Q_UNUSED(action)
-
     //FIXME this only works for internal drag and drops
     //FIXME Moving is *very* buggy
-
-    QModelIndex idx;
+    QModelIndex dropDestIndex;
+    bool isInsertBetweenOp = false;
     if(row == -1)
-        idx = parent;
+    {
+        dropDestIndex = parent;
+    }
     else
-        idx = index(row, column, parent);
+    {
+        isInsertBetweenOp = true;
+        dropDestIndex= index(row, column, parent);
+    }
 
-    KBookmark bk = bookmarkForIndex(idx);
-    QString addr = bk.address();
-    if(bk.isGroup())
-        addr += "/0";
+    KBookmark dropDestBookmark = bookmarkForIndex(dropDestIndex);
+    if (dropDestBookmark.isNull())
+    {
+        // Presumably an invalid index: assume we want to place this in the root bookmark
+        // folder.
+        dropDestBookmark = d->mRoot;
+    }
 
+    QString addr = dropDestBookmark.address();
+    if(dropDestBookmark.isGroup() && !isInsertBetweenOp)
+    {
+      addr += "/0";
+    }
+    // bookmarkForIndex(...) does not distinguish between the last item in the folder
+    // and the point *after* the last item in the folder (and its hard to see how to
+    // modify it so it does), so do the check here.
+    if (isInsertBetweenOp && row == dropDestBookmark.positionInParent() + 1)
+    {
+        // Attempting to insert underneath the last item in a folder; adjust the address.
+        addr = KBookmark::nextAddress(addr);
+    }
+
     if(action == Qt::CopyAction)
     {
         KEBMacroCommand * cmd = CmdGen::insertMimeSource("Copy", data, addr);
@@ -338,6 +360,10 @@
 
 KBookmark KBookmarkModel::bookmarkForIndex(const QModelIndex& index) const
 {
+    if (!index.isValid())
+    {
+      return KBookmark();
+    }
     return static_cast<TreeItem *>(index.internalPointer())->bookmark();
 }
 
Index: apps/keditbookmarks/bookmarklistview.cpp
===================================================================
--- apps/keditbookmarks/bookmarklistview.cpp	(.../tags/KDE/4.3.2/kdebase)	(wersja 1032821)
+++ apps/keditbookmarks/bookmarklistview.cpp	(.../branches/KDE/4.3/kdebase)	(wersja 1032821)
@@ -43,6 +43,7 @@
 
 }
 
+
 /* ----------- */
 
 BookmarkFolderView::BookmarkFolderView( BookmarkListView * view, QWidget * parent )
@@ -53,6 +54,7 @@
     setModel(mmodel);
     header()->setVisible(false);
     setRootIsDecorated(false);
+    setDropIndicatorShown(true);
     expandAll();
     setCurrentIndex( mmodel->index(0,0, QModelIndex()));
     
@@ -88,7 +90,6 @@
     return static_cast<KBookmarkModel *>(mmodel->sourceModel())->bookmarkForIndex(index);
 }
 
-
 /********/
 
 
@@ -173,15 +174,32 @@
 
 bool BookmarkFolderViewFilterModel::dropMimeData(const QMimeData *data, Qt::DropAction action, int row, int column, const QModelIndex &parent)
 {
-    // FIXME Probably bug in QT, send bug report
-    kDebug()<<"BookmarkFolderViewFilterModel::dropMimeData"<<endl;
-    QModelIndex idx;
+    QModelIndex dropDestProxyIndex;
+    bool isInsertBetweenOp = false;
     if(row == -1)
-        idx = parent;
+    {
+        dropDestProxyIndex = parent;
+    }
     else
-        idx = index(row, column, parent);
-    QModelIndex src = mapToSource(idx);
-    return sourceModel()->dropMimeData( data, action, -1, -1, src);
+    {
+        dropDestProxyIndex = index(row, column, parent);
+        isInsertBetweenOp = true;
+    }
+    QModelIndex dropDestIndex = mapToSource(dropDestProxyIndex);
+    if (!isInsertBetweenOp)
+    {
+        // Just dropping it onto dropDestIndex - ignore row and column.
+        return sourceModel()->dropMimeData( data, action, -1, -1, dropDestIndex);
+    }
+    else
+    {
+        // Dropping before dropDestIndex.  We want to keep row and column
+        // relative to the parent.
+        // I'm reasonably certain the parent must be valid in this case.  If you get a crash here - nag me!
+        Q_ASSERT(parent.isValid());
+        QModelIndex dropDestParentIndex = mapToSource(parent);
+        return sourceModel()->dropMimeData( data, action, dropDestIndex.row(), dropDestIndex.column(), dropDestParentIndex);
+    }
 }
 
 BookmarkFolderViewFilterModel::~BookmarkFolderViewFilterModel()
Index: runtime/kcontrol/emoticons/emoticonslist.cpp
===================================================================
Index: runtime/kcontrol/componentchooser/componentservices/kcm_filemanager.desktop
===================================================================
Index: runtime/phonon/platform_kde/phonon.notifyrc
===================================================================
Index: runtime/kioslave/info/info.cc
===================================================================
Index: runtime/nepomuk/services/storage/nepomukcore.cpp
===================================================================
Index: runtime/nepomuk/kcm/nepomukserverkcm.cpp
===================================================================
Index: runtime/l10n/jm/entry.desktop
===================================================================
Index: runtime/l10n/ru/entry.desktop
===================================================================
Index: runtime/l10n/tj/entry.desktop
===================================================================
Index: runtime/l10n/ge/entry.desktop
===================================================================
Index: runtime/plasma/scriptengines/javascript/plasma-scriptengine-applet-simple-javascript.desktop
===================================================================
Index: workspace/ksmserver/fadeeffect.cpp
===================================================================
Index: workspace/wallpapers/Atra_Dot/metadata.desktop
===================================================================
Index: workspace/solid/solid-actions-kcm/device-actions/solid-device-DvbInterface.desktop
===================================================================
Index: workspace/solid/networking/kded/networkstatus.desktop
===================================================================
Index: workspace/khotkeys/data/kde32b1.khotkeys
===================================================================
Index: workspace/kwin/effects/desktopgrid/desktopgrid.cpp
===================================================================
Index: workspace/kwin/effects/slideback/slideback.cpp
===================================================================
Index: workspace/kwin/effects/logout/logout.h
===================================================================
Index: workspace/kwin/effects/logout/logout.cpp
===================================================================
Index: workspace/kwin/utils.h
===================================================================
Index: workspace/kwin/rules.cpp
===================================================================
Index: workspace/kwin/client.cpp
===================================================================
Index: workspace/kwin/kwin.notifyrc
===================================================================
Index: workspace/kwin/geometry.cpp
===================================================================
Index: workspace/kwin/client.h
===================================================================
Index: workspace/kwin/utils.cpp
===================================================================
Index: workspace/kwin/activation.cpp
===================================================================
Index: workspace/plasma/dataengines/keystate/plasma-dataengine-keystate.desktop
===================================================================
Index: workspace/plasma/runners/locations/plasma-runner-locations.desktop
===================================================================
Index: workspace/plasma/scriptengines/ruby/examples/dataengines/time/metadata.desktop
===================================================================
