Index: apps/konqueror/settings/konq/globalpaths.cpp
===================================================================
--- apps/konqueror/settings/konq/globalpaths.cpp	(.../tags/KDE/4.3.4/kdebase)	(wersja 1058840)
+++ apps/konqueror/settings/konq/globalpaths.cpp	(.../branches/KDE/4.3/kdebase)	(wersja 1058840)
@@ -329,6 +329,12 @@
         return true;
     if (!QFile::exists(src.toLocalFile()))
         return true;
+    // Do not move $HOME! #193057
+    const QString translatedPath = translatePath(src.toLocalFile());
+    if (translatedPath == "$HOME" || translatedPath == "$HOME/") {
+        return true;
+    }
+
     m_ok = true;
     // TODO: check if the src dir is empty? Nothing to move, then...
 
Index: runtime/kioslave/info/kde-info2html
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/grec.desktop
===================================================================
Index: workspace/kcontrol/input/logitechmouse.cpp
===================================================================
Index: workspace/kcontrol/keys/keys.desktop
===================================================================
Index: workspace/kwin/effects/presentwindows/presentwindows.desktop
===================================================================
Index: workspace/kwin/effects/presentwindows/presentwindows_config.desktop
===================================================================
Index: workspace/plasma/applets/kickoff/simpleapplet/plasma-applet-simplelauncher.desktop
===================================================================
Index: workspace/plasma/applets/kickoff/applet/plasma-applet-launcher.desktop
===================================================================
