Index: apps/konqueror/src/konqmainwindow.cpp
===================================================================
--- apps/konqueror/src/konqmainwindow.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/konqueror/src/konqmainwindow.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -1174,10 +1174,12 @@
         if ( windowArgs.lowerWindow() || (QApplication::keyboardModifiers() & Qt::ShiftModifier))
            newtabsinfront = !newtabsinfront;
         const bool aftercurrentpage = KonqSettings::openAfterCurrentPage();
+        KonqOpenURLRequest req;
+        req.args = args;
+        req.browserArgs = browserArgs;
 
         // Can we use the standard way (openUrl), or do we need the part pointer immediately?
         if (!part) {
-            KonqOpenURLRequest req;
             req.newTab = true;
             req.newTabInFront = newtabsinfront;
             req.openAfterCurrentPage = aftercurrentpage;
@@ -1189,7 +1191,7 @@
             if (newtabsinfront)
                 m_pViewManager->showTab( newView );
 
-            openUrl( newView, url.isEmpty() ? KUrl("about:blank") : url, QString() );
+            openUrl( newView, url.isEmpty() ? KUrl("about:blank") : url, QString(), req );
             newView->setViewName( browserArgs.frameName );
 
             *part = newView->part();
@@ -2175,12 +2177,15 @@
 KonqView * KonqMainWindow::childView( KParts::ReadOnlyPart *callingPart, const QString &name, KParts::BrowserHostExtension *&hostExtension, KParts::ReadOnlyPart **part )
 {
     //kDebug() << "this=" << this << "looking for" << name;
+    QList<KonqView *> views = m_mapViews.values();
+    KonqView* callingView = m_mapViews.value(callingPart);
+    if (callingView) {
+        // Move the callingView in front of the list, in case of duplicate frame names (#133967)
+        if (views.removeAll(callingView))
+            views.prepend(callingView);
+    }
 
-  MapViews::ConstIterator it = m_mapViews.constBegin();
-  MapViews::ConstIterator end = m_mapViews.constEnd();
-  for (; it != end; ++it )
-  {
-    KonqView* view = it.value();
+  Q_FOREACH(KonqView* view, views) {
     QString viewName = view->viewName();
     //kDebug() << "       - viewName=" << viewName
     //          << "frame names:" << view->frameNames();
Index: apps/konqueror/src/konqtabs.cpp
===================================================================
--- apps/konqueror/src/konqtabs.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/konqueror/src/konqtabs.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -266,7 +266,9 @@
 void KonqFrameTabs::setTitle( const QString &title , QWidget* sender)
 {
   // kDebug(1202) << "KonqFrameTabs::setTitle( " << title << " , " << sender << " )";
-  setTabText( indexOf( sender ), title );
+  // Make sure that '&' is displayed correctly
+  QString tabText( title );
+  setTabText( indexOf( sender ), tabText.replace('&', "&&") );
 }
 
 void KonqFrameTabs::setTabIcon( const KUrl &url, QWidget* sender )
Index: apps/konqueror/settings/kio/lanbrowser.desktop
===================================================================
--- apps/konqueror/settings/kio/lanbrowser.desktop	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/konqueror/settings/kio/lanbrowser.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -139,7 +139,7 @@
 Comment[pl]=Konfiguracja przeglądania sieci lokalnej (współdzielone pliki i drukarki)
 Comment[pt]=Configurar a navegação na rede local para as pastas e impressoras partilhadas
 Comment[pt_BR]=Configurar a rede local navegando por pastas compartilhadas e impressoras
-Comment[ro]=Configurează răsfoirea rețelei locale pentru dosare și imprimante pertajate
+Comment[ro]=Configurează răsfoirea rețelei locale pentru dosare și imprimante partajate
 Comment[ru]=Настройка обзора общих папок и принтеров в локальной сети
 Comment[sk]=Konfigurácia prehliadania zdieľaných priečinkov a tlačiarní v lokálnej sieti
 Comment[sl]=Nastavitev brskanja po mapah in tiskalnikih, ki so v souporabi v krajevnem omrežju
Index: apps/konqueror/settings/konqhtml/khtml_appearance.desktop
===================================================================
--- apps/konqueror/settings/konqhtml/khtml_appearance.desktop	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/konqueror/settings/konqhtml/khtml_appearance.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -96,6 +96,7 @@
 
 
 Comment=Configure how to display web pages
+Comment[bg]=Настройване как да се показват уеб страниците
 Comment[ca]=Configura com visualitzar les pàgines web
 Comment[cs]=Nastavit zobrazení webových stránek
 Comment[da]=Indstil hvordan netsider skal vises
Index: apps/kfind/kfwin.cpp
===================================================================
--- apps/kfind/kfwin.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/kfind/kfwin.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -80,7 +80,7 @@
   setText(5, matchingLine);
 
   // put the icon into the leftmost column
-  setPixmap(0, item.pixmap(16));
+  setPixmap(0, KIcon(item.iconName()).pixmap(16,16) );
 }
 
 KfFileLVI::~KfFileLVI()
Index: apps/kfind/kquery.cpp
===================================================================
--- apps/kfind/kquery.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/kfind/kquery.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -79,11 +79,9 @@
   if(m_useLocate) //use "locate" instead of the internal search method
   {
     m_url.cleanPath();
-    processLocate->clearProgram();
-    *processLocate << "locate";
-    *processLocate << m_url.path( KUrl::AddTrailingSlash ).toLatin1();
     bufferLocate.clear();
-    processLocate->setNextOpenMode(QIODevice::Text);
+    processLocate->clearProgram();
+    processLocate->setProgram( "locate", QStringList() << m_url.path( KUrl::AddTrailingSlash ) );
     processLocate->setOutputChannelMode(KProcess::SeparateChannels);
     processLocate->start();
     return;
@@ -501,7 +499,7 @@
 
   QString str = QString::fromLocal8Bit(bufferLocate);
   bufferLocate.clear();
-  slotListEntries(str.split('\n'));
+  slotListEntries(str.split('\n',QString::SkipEmptyParts));
   emit result(0);
 }
 
Index: apps/kfind/kfinddlg.cpp
===================================================================
--- apps/kfind/kfinddlg.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/kfind/kfinddlg.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -74,7 +74,9 @@
 
   connect(win ,SIGNAL(resultSelected(bool)),
 	  this,SIGNAL(resultSelected(bool)));
-
+      
+  connect( this, SIGNAL(closeClicked()), this, SLOT(finishAndClose()) );
+  
   query = new KQuery(frame);
   connect(query, SIGNAL(addFile(const KFileItem&,const QString&)),
 	  SLOT(addFile(const KFileItem&,const QString&)));
@@ -88,9 +90,11 @@
    stopSearch();
 }
 
-void KfindDlg::closeEvent(QCloseEvent *)
+void KfindDlg::finishAndClose()
 {
-   slotButtonClicked(KDialog::Close);
+  //Stop the current search and closes the dialog
+  stopSearch();
+  close();
 }
 
 void KfindDlg::setProgressMsg(const QString &msg)
Index: apps/kfind/kfinddlg.h
===================================================================
--- apps/kfind/kfinddlg.h	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/kfind/kfinddlg.h	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -34,7 +34,6 @@
   void setProgressMsg(const QString &);
 
 private:
-  void closeEvent(QCloseEvent *);
   /*Return a QStringList of all subdirs of d*/
   QStringList getAllSubdirs(QDir d);
 
@@ -50,6 +49,8 @@
   void slotDeleteItem(const QString&);
   void slotNewItems( const QString&  );
 
+  void finishAndClose();
+  
 Q_SIGNALS:
   void haveResults(bool);
   void resultSelected(bool);
Index: apps/dolphin/src/columnviewsettingspage.h
===================================================================
--- apps/dolphin/src/columnviewsettingspage.h	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/dolphin/src/columnviewsettingspage.h	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -53,6 +53,12 @@
     void loadSettings();
 
 private:
+    enum
+    {
+        BaseTextWidth = 200,
+        TextInc = 50
+    };
+    
     IconSizeGroupBox* m_iconSizeGroupBox;
     DolphinFontRequester* m_fontRequester;
     KComboBox* m_textWidthBox;
Index: apps/dolphin/src/dolphinview.cpp
===================================================================
--- apps/dolphin/src/dolphinview.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/dolphin/src/dolphinview.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -1095,18 +1095,20 @@
 
 void DolphinView::restoreCurrentItem()
 {
-    const QModelIndex dirIndex = m_dolphinModel->indexForUrl(m_currentItemUrl);
-    if (dirIndex.isValid()) {
-        const QModelIndex proxyIndex = m_proxyModel->mapFromSource(dirIndex);
-        QAbstractItemView* view = itemView();
-        const bool clearSelection = !hasSelection();
-        view->setCurrentIndex(proxyIndex);
-        if (clearSelection) {
-            view->clearSelection();
+    if (!m_currentItemUrl.isEmpty()) {
+        const QModelIndex dirIndex = m_dolphinModel->indexForUrl(m_currentItemUrl);
+        if (dirIndex.isValid()) {
+            const QModelIndex proxyIndex = m_proxyModel->mapFromSource(dirIndex);
+            QAbstractItemView* view = itemView();
+            const bool clearSelection = !hasSelection();
+            view->setCurrentIndex(proxyIndex);
+            if (clearSelection) {
+                view->clearSelection();
+            }
         }
+        m_currentItemUrl.clear();
     }
 }
-
 void DolphinView::enterDir(const QModelIndex& index, QAbstractItemView* view)
 {
     // Deleting a view that is the root of a drag operation is not allowed, otherwise
Index: apps/dolphin/src/dolphinpart.cpp
===================================================================
--- apps/dolphin/src/dolphinpart.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/dolphin/src/dolphinpart.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -331,10 +331,7 @@
 {
     // TODO: Check issue N176832 for the missing QAIV signal; task 177399 - maybe this code
     // should be moved into DolphinPart::slotItemTriggered()
-    KFileItem item(S_IFDIR, (mode_t)-1, url);
-    KParts::OpenUrlArguments args;
-    args.setMimeType(item.mimetype());
-    emit m_extension->createNewWindow(url, args);
+    emit m_extension->createNewWindow(url);
 }
 
 void DolphinPart::slotOpenContextMenu(const KFileItem& _item, const KUrl&)
Index: apps/dolphin/src/dolphinpart.desktop
===================================================================
--- apps/dolphin/src/dolphinpart.desktop	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/dolphin/src/dolphinpart.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -178,7 +178,7 @@
 Name[ar]=تفاصيل
 Name[as]=বিৱৰণ
 Name[be@latin]=Detali
-Name[bg]=Подробности
+Name[bg]=Подробно
 Name[bn]=বিস্তারিত
 Name[bn_IN]=বিবরণ
 Name[ca]=Detalls
Index: apps/dolphin/src/infosidebarpage.cpp
===================================================================
--- apps/dolphin/src/infosidebarpage.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/dolphin/src/infosidebarpage.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -541,7 +541,8 @@
     m_nameLabel->setFont(font);
     m_nameLabel->setAlignment(Qt::AlignHCenter);
     m_nameLabel->setSizePolicy(QSizePolicy::Preferred, QSizePolicy::Fixed);
-
+    m_nameLabel->setMaximumWidth(KIconLoader::SizeEnormous);
+    
     // preview
     m_preview = new PixmapViewer(this);
     m_preview->setMinimumWidth(KIconLoader::SizeEnormous + KIconLoader::SizeMedium);
Index: apps/dolphin/src/columnviewsettingspage.cpp
===================================================================
--- apps/dolphin/src/columnviewsettingspage.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/dolphin/src/columnviewsettingspage.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -113,9 +113,8 @@
     settings->setItalicFont(font.italic());
     settings->setFontWeight(font.weight());
 
-    // TODO:
-    //const int columnWidth = 150 + (m_columnWidthSlider->value() * 50);
-    //settings->setColumnWidth(columnWidth);
+    const int columnWidth = BaseTextWidth + (m_textWidthBox->currentIndex() * TextInc);
+    settings->setColumnWidth(columnWidth);
     
     settings->writeConfig();
 }
@@ -150,8 +149,7 @@
         m_fontRequester->setCustomFont(font);
     }
 
-    // TODO:
-    //m_columnWidthSlider->setValue((settings->columnWidth() - 150) / 50);
+    m_textWidthBox->setCurrentIndex((settings->columnWidth() - BaseTextWidth) / TextInc);
 }
 
 #include "columnviewsettingspage.moc"
Index: apps/dolphin/src/dolphinmodel.cpp
===================================================================
--- apps/dolphin/src/dolphinmodel.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/dolphin/src/dolphinmodel.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -248,7 +248,8 @@
                 Q_ASSERT(false);
             }
         } else {
-            if (daysDistance <= (currentDate.day() + modifiedDate.daysInMonth())) {
+            const QDate lastMonthDate = currentDate.addMonths(-1);
+            if  (lastMonthDate.year() == modifiedDate.year() && lastMonthDate.month() == modifiedDate.month()) {
                 if (daysDistance == 1) {
                     retString = i18nc("@title:group Date: %B is full month name in current locale, and %Y is full year number", "Yesterday (%B, %Y)");
                 } else if (daysDistance <= 7) {
Index: apps/dolphin/src/dolphincolumnwidget.cpp
===================================================================
--- apps/dolphin/src/dolphincolumnwidget.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/dolphin/src/dolphincolumnwidget.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -517,6 +517,8 @@
 void DolphinColumnWidget::deactivate()
 {
     clearFocus();
+    disconnect(this, SIGNAL(clicked(const QModelIndex&)),
+               m_view->m_controller, SLOT(requestTab(const QModelIndex&)));
 
     // TODO: Connecting to the signal 'activated()' is not possible, as kstyle
     // does not forward the single vs. doubleclick to it yet (KDE 4.1?). Hence it is
Index: apps/lib/konq/konq_menuactions.cpp
===================================================================
--- apps/lib/konq/konq_menuactions.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/lib/konq/konq_menuactions.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -185,7 +185,7 @@
     const KFileItemList items = d->m_info.items();
     const KFileItem firstItem = items.first();
     const QString protocol = firstItem.url().protocol(); // assumed to be the same for all items
-    const bool isLocal = firstItem.url().isLocalFile();
+    const bool isLocal = !firstItem.localPath().isEmpty();
     const bool isSingleLocal = items.count() == 1 && isLocal;
     const KUrl::List urlList = d->m_info.urlList();
 
@@ -195,8 +195,8 @@
     if (isSingleLocal && d->m_info.mimeType() == "application/x-desktop") // .desktop file
     {
         // get builtin services, like mount/unmount
-        s.builtin = KDesktopFileActions::builtinServices(firstItem.url());
-        const QString path = firstItem.url().path();
+        const QString path = firstItem.localPath();
+        s.builtin = KDesktopFileActions::builtinServices(path);
         KDesktopFile desktopFile(path);
         KConfigGroup cfg = desktopFile.desktopGroup();
         const QString priority = cfg.readEntry("X-KDE-Priority");
@@ -216,7 +216,7 @@
 
     // first check the .directory if this is a directory
     if (d->m_info.isDirectory() && isSingleLocal) {
-        QString dotDirectoryFile = firstItem.url().path(KUrl::AddTrailingSlash).append(".directory");
+        QString dotDirectoryFile = KUrl::fromPath(firstItem.localPath()).path(KUrl::AddTrailingSlash).append(".directory");
         if (QFile::exists(dotDirectoryFile)) {
             const KDesktopFile desktopFile(  dotDirectoryFile );
             const KConfigGroup cfg = desktopFile.desktopGroup();
Index: apps/kinfocenter/pci/kcm_pci.cpp
===================================================================
--- apps/kinfocenter/pci/kcm_pci.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/kinfocenter/pci/kcm_pci.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -80,6 +80,11 @@
 void KCMPci::load() {
 	kDebug() << "Loading PCI information..." << endl;
 	GetInfo_PCI(tree);
+
+	//Resize the column width to the maximum needed
+	tree->expandAll();
+	tree->resizeColumnToContents( 0 );
+	tree->collapseAll();
 }
 
 QString KCMPci::quickHelp() const {
Index: apps/kinfocenter/base/info_fbsd.cpp
===================================================================
--- apps/kinfocenter/base/info_fbsd.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/kinfocenter/base/info_fbsd.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -89,7 +89,7 @@
 		new QTreeWidgetItem(tree, list);
 
 		/* Clean up after ourselves, this time I mean it ;-) */
-		delete buf;
+		delete[] buf;
 	}
 
 	return true;
Index: apps/plasma/applets/folderview/proxymodel.cpp
===================================================================
--- apps/plasma/applets/folderview/proxymodel.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/plasma/applets/folderview/proxymodel.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -25,7 +25,7 @@
 
 
 ProxyModel::ProxyModel(QObject *parent)
-    : QSortFilterProxyModel(parent), m_sortDirsFirst(true)  
+    : QSortFilterProxyModel(parent), m_filterMode(NoFilter), m_sortDirsFirst(true)
 {
     setSupportedDragActions(Qt::CopyAction | Qt::MoveAction | Qt::LinkAction);
 }
Index: apps/plasma/applets/folderview/dialog.h
===================================================================
--- apps/plasma/applets/folderview/dialog.h	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/plasma/applets/folderview/dialog.h	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -35,14 +35,14 @@
 class Dialog : public QWidget
 {
 public:
-    Dialog(QWidget *parent = 0, Qt::WindowFlags f = Qt::Window);
+    Dialog(QWidget *parent = 0);
     ~Dialog();
 
     void setGraphicsWidget(QGraphicsWidget *widget);
     void show(Plasma::Applet *applet);
 
 protected:
-    void focusOutEvent(QFocusEvent *event);
+    void mousePressEvent(QMouseEvent *event);
     void resizeEvent(QResizeEvent *event);
     void paintEvent(QPaintEvent *event);
 
Index: apps/plasma/applets/folderview/dialog.cpp
===================================================================
--- apps/plasma/applets/folderview/dialog.cpp	(.../tags/KDE/4.2.2/kdebase)	(wersja 956243)
+++ apps/plasma/applets/folderview/dialog.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 956243)
@@ -36,12 +36,12 @@
 #endif
 
 
-Dialog::Dialog(QWidget *parent, Qt::WindowFlags f)
-    : QWidget(parent, f), m_widget(0)
+Dialog::Dialog(QWidget *parent)
+    : QWidget(parent, Qt::Popup), m_widget(0)
 {
-    setWindowFlags(Qt::Popup | Qt::WindowStaysOnTopHint);
-    
 #ifdef Q_WS_X11
+    setAttribute(Qt::WA_X11NetWmWindowTypeDropDownMenu);
+
     if (!QX11Info::isCompositingManagerRunning()) {
         setAttribute(Qt::WA_NoSystemBackground);
     }
@@ -168,9 +168,10 @@
     m_background->paintFrame(&p);
 }
 
-void Dialog::focusOutEvent(QFocusEvent *event)
+void Dialog::mousePressEvent(QMouseEvent *event)
 {
-    Q_UNUSED(event)
-    hide();
+    if (!rect().contains(event->pos())) {
+        hide();
+    }
 }
 
Index: runtime/knotify/notifybyktts.cpp
===================================================================
Index: runtime/knotify/notifybyktts.h
===================================================================
Index: runtime/knotify/notifybyexecute.cpp
===================================================================
Index: runtime/knotify/notifybypopup.cpp
===================================================================
Index: runtime/kcontrol/locale/localetime.cpp
===================================================================
Index: runtime/phonon/kcm/xine/kcm_phononxine.desktop
===================================================================
Index: runtime/platforms/win/kwinstartmenu/winstartmenu.desktop
===================================================================
Index: runtime/kioslave/info/info.cc
===================================================================
Index: runtime/kioslave/info/Messages.sh
===================================================================
Index: runtime/kstyles/themes/qtwindows.themerc
===================================================================
Index: runtime/kstyles/oxygen/oxygen.cpp
===================================================================
Index: runtime/kstyles/oxygen/CMakeLists.txt
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/technoratitags.desktop
===================================================================
Index: workspace/kcontrol/randr/krandrtray.desktop
===================================================================
Index: workspace/kcontrol/randr/randr.desktop
===================================================================
Index: workspace/kcontrol/keys/keys.desktop
===================================================================
Index: workspace/solid/kcm/kcm_solid.desktop
===================================================================
Index: workspace/powerdevil/daemon/PollSystemLoader.cpp
===================================================================
Index: workspace/powerdevil/daemon/PowerDevilDaemon.cpp
===================================================================
Index: workspace/powerdevil/daemon/PollSystemLoader.h
===================================================================
Index: workspace/powerdevil/daemon/PowerDevilDaemon.h
===================================================================
Index: workspace/powerdevil/kcmodule/PowerDevilKCM.cpp
===================================================================
Index: workspace/powerdevil/kcmodule/powerdevilconfig.desktop
===================================================================
Index: workspace/powerdevil/powerdevil.notifyrc
===================================================================
Index: workspace/krunner/interfaces/default/interface.cpp
===================================================================
Index: workspace/krunner/interfaces/default/resultscene.cpp
===================================================================
Index: workspace/libs/kephal/xrandr12/randrcrtc.cpp
===================================================================
Index: workspace/libs/taskmanager/task.cpp
===================================================================
Index: workspace/kscreensaver/libkscreensaver/kscreensaver.h
===================================================================
Index: workspace/kscreensaver/libkscreensaver/kscreensaver.cpp
===================================================================
Index: workspace/khotkeys/data/kde32b1.khotkeys
===================================================================
Index: workspace/kwin/effects/taskbarthumbnail.desktop
===================================================================
Index: workspace/kwin/effects/magiclamp.cpp
===================================================================
Index: workspace/kwin/effects/magiclamp.h
===================================================================
Index: workspace/kwin/effects/desktopgrid.desktop
===================================================================
Index: workspace/kwin/main.cpp
===================================================================
Index: workspace/kdm/kfrontend/sessions/icewm.desktop
===================================================================
Index: workspace/ksysguard/ksysguardd/Command.c
===================================================================
Index: workspace/plasma/dataengines/powermanagement/plasma-dataengine-powermanagement.desktop
===================================================================
Index: workspace/plasma/shells/desktop/panelview.cpp
===================================================================
Index: workspace/plasma/runners/locations/locationrunner.cpp
===================================================================
Index: workspace/plasma/tools/desktopthemedetails/desktopthemedetails.desktop
===================================================================
Index: workspace/plasma/applets/calendar/calendar.h
===================================================================
Index: workspace/plasma/applets/calendar/calendar.cpp
===================================================================
Index: workspace/plasma/applets/system-monitor/net.cpp
===================================================================
Index: workspace/plasma/applets/battery/battery.cpp
===================================================================
Index: workspace/plasma/applets/kickoff/ui/launcher.cpp
===================================================================
Index: workspace/plasma/scriptengines/webkit/plasma-packagestructure-dashboard.desktop
===================================================================
Index: workspace/plasma/scriptengines/webkit/plasma-scriptengine-applet-dashboard.desktop
===================================================================
Index: workspace/plasma/scriptengines/javascript/tests/script-digital-clock/metadata.desktop
===================================================================
