Index: apps/kappfinder/apps/System/Terminal/vmstat.desktop
===================================================================
--- apps/kappfinder/apps/System/Terminal/vmstat.desktop	(.../tags/KDE/4.2.1/kdebase)	(wersja 935993)
+++ apps/kappfinder/apps/System/Terminal/vmstat.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 935993)
@@ -43,7 +43,7 @@
 GenericName[gu]=વર્ચ્યુઅલ મેમરી આંકડાઓ
 GenericName[he]=סטטיסטיקה לגבי הזיכרון הווירטואלי
 GenericName[hi]=आभासी मेमोरी आंकड़े
-GenericName[hne]=आभासी मेमोरी आंकड़े
+GenericName[hne]=आभासी मेमोरी आंकड़ा
 GenericName[hr]=Statistike o virtualnoj memoriji
 GenericName[hsb]=Statistika wo wirtuelnym pomjatku
 GenericName[hu]=Memóriastatisztika
Index: apps/konqueror/src/konqbookmarkbar.cpp
===================================================================
--- apps/konqueror/src/konqbookmarkbar.cpp	(.../tags/KDE/4.2.1/kdebase)	(wersja 935993)
+++ apps/konqueror/src/konqbookmarkbar.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 935993)
@@ -301,10 +301,21 @@
 {
     KBookmarkActionInterface * action = dynamic_cast<KBookmarkActionInterface *>( m_toolBar->actionAt(pos) );
     if(!action)
-        return;
-    KMenu * menu = new KonqBookmarkContextMenu(action->bookmark(), m_pManager, m_pOwner);
-    menu->setAttribute(Qt::WA_DeleteOnClose);
-    menu->popup(m_toolBar->mapToGlobal(pos));
+    {
+        //Show default (ktoolbar) menu
+        m_toolBar->setContextMenuPolicy( Qt::DefaultContextMenu );
+        //Recreate event with the same position
+        QContextMenuEvent evt( QContextMenuEvent::Other, pos );
+        QCoreApplication::sendEvent( m_toolBar, &evt );
+        //Reassign custom context menu
+        m_toolBar->setContextMenuPolicy( Qt::CustomContextMenu );
+    }
+    else
+    {
+        KMenu * menu = new KonqBookmarkContextMenu(action->bookmark(), m_pManager, m_pOwner);
+        menu->setAttribute(Qt::WA_DeleteOnClose);
+        menu->popup(m_toolBar->mapToGlobal(pos));
+    }
 }
 
 // TODO    *** drop improvements ***
Index: apps/kinfocenter/kinfocenter.desktop
===================================================================
--- apps/kinfocenter/kinfocenter.desktop	(.../tags/KDE/4.2.1/kdebase)	(wersja 935993)
+++ apps/kinfocenter/kinfocenter.desktop	(.../branches/KDE/4.2/kdebase)	(wersja 935993)
@@ -103,7 +103,7 @@
 GenericName[pt]=Centro de Informações
 GenericName[pt_BR]=Centro de informações
 GenericName[ro]=Centru de informații
-GenericName[ru]=Центр информации
+GenericName[ru]=Информация о системе
 GenericName[se]=Diehtoguovddáš
 GenericName[sk]=Informačné centrum
 GenericName[sl]=Informacijsko središče
Index: apps/konsole/src/TerminalDisplay.h
===================================================================
--- apps/konsole/src/TerminalDisplay.h	(.../tags/KDE/4.2.1/kdebase)	(wersja 935993)
+++ apps/konsole/src/TerminalDisplay.h	(.../branches/KDE/4.2/kdebase)	(wersja 935993)
@@ -563,9 +563,6 @@
     //     - Other characters (returns the input character)
     QChar charClass(QChar ch) const;
 
-    // Returns true upon a word boundary
-    bool isCharBoundary(QChar ch) const;
-
     void clearImage();
 
     void mouseTripleClickEvent(QMouseEvent* ev);
Index: apps/konsole/src/TerminalDisplay.cpp
===================================================================
--- apps/konsole/src/TerminalDisplay.cpp	(.../tags/KDE/4.2.1/kdebase)	(wersja 935993)
+++ apps/konsole/src/TerminalDisplay.cpp	(.../branches/KDE/4.2/kdebase)	(wersja 935993)
@@ -2155,11 +2155,12 @@
   _wordSelectionMode = true;
 
   // find word boundaries...
+  QChar selClass = charClass(_image[i].character);
   {
      // find the start of the word
      int x = bgnSel.x();
      while ( ((x>0) || (bgnSel.y()>0 && (_lineProperties[bgnSel.y()-1] & LINE_WRAPPED) )) 
-                     && !isCharBoundary(_image[i-1].character) )
+                     && charClass(_image[i-1].character) == selClass )
      {  
        i--; 
        if (x>0) 
@@ -2178,7 +2179,7 @@
      i = loc( endSel.x(), endSel.y() );
      x = endSel.x();
      while( ((x<_usedColumns-1) || (endSel.y()<_usedLines-1 && (_lineProperties[endSel.y()] & LINE_WRAPPED) )) 
-                     && !isCharBoundary(_image[i+1].character) )
+                     && charClass(_image[i+1].character) == selClass )
      { 
          i++; 
          if (x<_usedColumns-1) 
@@ -2332,16 +2333,7 @@
   return QWidget::focusNextPrevChild( next );
 }
 
-// Returns true upon a word boundary
-// TODO determine if the below charClass() is actually required
-bool TerminalDisplay::isCharBoundary(QChar qch) const
-{
-    if ( _wordCharacters.contains(qch, Qt::CaseInsensitive) ) return true;
-    if ( qch.isSpace() ) return true;
 
-    return false;
-}
-
 QChar TerminalDisplay::charClass(QChar qch) const
 {
     if ( qch.isSpace() ) return ' ';
Index: runtime/cmake/modules/FindPulseAudio.cmake
===================================================================
Index: runtime/knotify/kde.notifyrc
===================================================================
Index: runtime/desktoptheme/metadata.desktop
===================================================================
Index: runtime/khelpcenter/kcmhelpcenter.desktop
===================================================================
Index: runtime/kioslave/trash/trash.protocol
===================================================================
Index: runtime/l10n/ae/entry.desktop
===================================================================
Index: runtime/l10n/au/entry.desktop
===================================================================
Index: workspace/ConfigureChecks.cmake
===================================================================
Index: workspace/klipper/klipperrc.desktop
===================================================================
Index: workspace/powerdevil/powerdevil.notifyrc
===================================================================
Index: workspace/libs/solid/CMakeLists.txt
===================================================================
Index: workspace/khotkeys/kcm_hotkeys/hotkeys_tree_view.cpp
===================================================================
Index: workspace/khotkeys/data/konqueror_gestures_kde321.khotkeys
===================================================================
Index: workspace/khotkeys/data/kde32b1.khotkeys
===================================================================
Index: workspace/kwin/effects/blur.desktop
===================================================================
Index: workspace/kwin/popupinfo.cpp
===================================================================
Index: workspace/kwin/kwin.notifyrc
===================================================================
Index: workspace/kdm/kfrontend/sessions/xfce4.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/ctwm.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/oroborus.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/9wm.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/ude.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/xfce.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/waimea.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/w9wm.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/cde.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/vtwm.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/sawfish.desktop
===================================================================
Index: workspace/plasma/dataengines/CMakeLists.txt
===================================================================
Index: workspace/plasma/shells/desktop/panelview.cpp
===================================================================
Index: workspace/plasma/containments/desktop/desktop.cpp
===================================================================
Index: workspace/plasma/applets/trash/plasma-applet-trash.desktop
===================================================================
Index: workspace/plasma/applets/devicenotifier/test-predicate-openinwindow.desktop
===================================================================
Index: workspace/plasma/applets/quicklaunch/plasma-applet-quicklaunch.desktop
===================================================================
Index: workspace/plasma/applets/calendar/calendar.h
===================================================================
Index: workspace/plasma/applets/calendar/calendar.cpp
===================================================================
Index: workspace/plasma/applets/kickoff/ui/launcher.cpp
===================================================================
Index: workspace/plasma/applets/CMakeLists.txt
===================================================================
Index: workspace/plasma/scriptengines/ruby/ChangeLog
===================================================================
Index: workspace/plasma/scriptengines/ruby/applet.rb
===================================================================
Index: workspace/plasma/scriptengines/google_gadgets/plasma-scriptengine-googlegadgets.desktop
===================================================================
Index: workspace/plasma/scriptengines/google_gadgets/plasma-packagestructure-googlegadgets.desktop
===================================================================
Index: workspace/plasma/scriptengines/qedjescript/plasma-appletscript-qedje.desktop
===================================================================
Index: workspace/plasma/scriptengines/qedjescript/plasma-packagestructure-qedje.desktop
===================================================================
