Index: apps/konqueror/src/konqtabs.h
===================================================================
--- apps/konqueror/src/konqtabs.h	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/konqueror/src/konqtabs.h	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -143,7 +143,6 @@
   NewTabToolButton* m_leftWidget;
   bool m_permanentCloseButtons;
   bool m_alwaysTabBar;
-  bool m_MouseMiddleClickClosesTab;
   QMap<QString,QAction*> m_popupActions;
 };
 
Index: apps/konqueror/src/konqcombo.cpp
===================================================================
--- apps/konqueror/src/konqcombo.cpp	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/konqueror/src/konqcombo.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -869,6 +869,8 @@
                 const bool blocked = completionbox->blockSignals(true);
                 completionbox->setCurrentItem(matchedItem);
                 completionbox->blockSignals(blocked);
+            } else {
+                completionbox->setCurrentRow(-1);
             }
         }
         else { // completion box not visible yet -> show it
Index: apps/konqueror/src/konqtabs.cpp
===================================================================
--- apps/konqueror/src/konqtabs.cpp	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/konqueror/src/konqtabs.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -74,8 +74,6 @@
   m_pActiveChild = 0L;
   m_pViewManager = viewManager;
 
-  m_MouseMiddleClickClosesTab = KonqSettings::mouseMiddleClickClosesTab();
-
   m_permanentCloseButtons = KonqSettings::permanentCloseButton();
   if (m_permanentCloseButtons) {
     setTabsClosable( true );
@@ -379,7 +377,7 @@
 
 void KonqFrameTabs::slotMouseMiddleClick(QWidget *w)
 {
-    if (m_MouseMiddleClickClosesTab) {
+    if (KonqSettings::mouseMiddleClickClosesTab()) {
         slotCloseRequest(w);
     } else {
         KUrl filteredURL(KonqMisc::konqFilteredURL(this, QApplication::clipboard()->text(QClipboard::Selection)));
Index: apps/konqueror/settings/kio/kcookiespolicies.cpp
===================================================================
--- apps/konqueror/settings/kio/kcookiespolicies.cpp	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/konqueror/settings/kio/kcookiespolicies.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -191,9 +191,9 @@
     if ( !handleDuplicate(domain, advice) )
     {
       const char* strAdvice = KCookieAdvice::adviceToStr(advice);
-      QTreeWidgetItem* index = new QTreeWidgetItem (dlg->lvDomainPolicy,
-                                                QStringList()<< domain << i18n(strAdvice));
-      m_pDomainPolicy.insert (index, strAdvice);
+      QTreeWidgetItem* item = new QTreeWidgetItem (dlg->lvDomainPolicy,
+                                                   QStringList()<< domain << i18n(strAdvice));
+      m_pDomainPolicy.insert (item, strAdvice);
       configChanged();
     }
   }
@@ -395,12 +395,14 @@
   group.writeEntry("CookieGlobalAdvice", advice);
 
   QStringList domainConfig;
-  QTreeWidgetItem *at = dlg->lvDomainPolicy->topLevelItem(0);
-
-  while( at )
-  {
-    domainConfig.append(QString::fromLatin1("%1:%2").arg(QString(tolerantToAce(at->text(0)))).arg(m_pDomainPolicy[at]));
-    at = dlg->lvDomainPolicy->itemBelow(at);
+  QMapIterator<QTreeWidgetItem*, const char*> it (m_pDomainPolicy);
+  while (it.hasNext()) {
+      it.next();
+      QTreeWidgetItem *item = it.key();
+      QString policy = tolerantToAce(item->text(0));
+      policy += QLatin1Char(':');
+      policy += QLatin1String(it.value());
+      domainConfig << policy;
   }
 
   group.writeEntry("CookieDomainAdvice", domainConfig);
Index: apps/konqueror/settings/kio/uasproviders/safari40.desktop
===================================================================
--- apps/konqueror/settings/kio/uasproviders/safari40.desktop	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/konqueror/settings/kio/uasproviders/safari40.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -12,6 +12,7 @@
 Name[en_GB]=UADescription (Safari 4.0 on MacOS X)
 Name[es]=UADescription (Safari 4.0 en MacOS X)
 Name[et]=UADescription (Safari 4.0 MacOS X platvormil)
+Name[eu]=UA azalpena (Safari 4.0 MacOS X-en)
 Name[fr]=UADescription (Safari 4.0 sous MacOS X)
 Name[fy]=UADescription (Safari 4.0 op MacOS X)
 Name[ga]=UADescription (Safari 4.0 ar MacOS X)
@@ -34,6 +35,7 @@
 Name[nb]=Brukeragent (Safari 4.0 på MacOS X)
 Name[nds]=UA-Beschrieven (Safari 4.0 op MacOS X)
 Name[nl]=GA-beschrijving (Safari 4.0 op MacOS X)
+Name[nn]=Brukaragent (Safari 4.0 på MacOS X)
 Name[pa]=UADescription (MacOS X ਉੱਤੇ ਸਫ਼ਾਰੀ ੪.੦)
 Name[pl]=UADescription (Safari 4.0 w MacOS X)
 Name[pt]=UADescription (Safari 4.0 no MacOS X)
Index: apps/konqueror/settings/kio/uasproviders/firefox36oncurrent.desktop
===================================================================
--- apps/konqueror/settings/kio/uasproviders/firefox36oncurrent.desktop	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/konqueror/settings/kio/uasproviders/firefox36oncurrent.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -12,6 +12,7 @@
 Name[en_GB]=UADescription (Firefox 3.6 on current)
 Name[es]=UADescription (Firefox 3.6 en actual)
 Name[et]=UADescription (Selles masinas töötav Firefox 3.6)
+Name[eu]=UA azalpena (Firefox 3.6 unekoan)
 Name[fr]=UADescription (Firefox 3.6 sous le système actuel)
 Name[fy]=UADescription (Firefox 3.6 op aktive)
 Name[ga]=UADescription (Firefox 3.6 ar an gcóras reatha)
@@ -34,6 +35,7 @@
 Name[nb]=Brukeragent (Firefox 3.6 herifra)
 Name[nds]=UA-Beschrieven (Firefox 3.6 op dit Systeem)
 Name[nl]=UADescription (Firefox 3.6 op huidige)
+Name[nn]=Brukaragent (Firefox 3.6 herifrå)
 Name[pa]=UADescription (ਮੌਜੂਦਾ ਉੱਤੇ ਫਾਇਰਫਾਕਸ ੩.੬)
 Name[pl]=UADescription (Firefox 3.6, bieżący)
 Name[pt]=UADescription (Firefox 3.6 no sistema actual)
Index: apps/konqueror/settings/kio/uasproviders/chrome50oncurrent.desktop
===================================================================
--- apps/konqueror/settings/kio/uasproviders/chrome50oncurrent.desktop	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/konqueror/settings/kio/uasproviders/chrome50oncurrent.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -12,6 +12,7 @@
 Name[en_GB]=UADescription (Google Chrome 5.0)
 Name[es]=UADescription (Google Chrome 5.0)
 Name[et]=UADescription (Google Chrome 5.0)
+Name[eu]=UA azalpena (Google Chrome 5.0)
 Name[fr]=UADescription (Google Chrome 5.0)
 Name[fy]=UADescription (Google Chrome 5.0)
 Name[ga]=UADescription (Google Chrome 5.0)
@@ -34,6 +35,7 @@
 Name[nb]=Brukeragent (Google Chrome 5.0)
 Name[nds]=UA-Beschrieven (Google Chrome 5.0)
 Name[nl]=GA-beschrijving (Google Chrome 5.0)
+Name[nn]=Brukaragent (Google Chrome 5.0)
 Name[pa]=UADescription (ਗੂਗਲ ਕਰੋਮ ੫.੦)
 Name[pl]=UADescription (Google Chrome 5.0)
 Name[pt]=UADescription (Google Chrome 5.0)
Index: apps/konqueror/settings/kio/uasproviders/firefox30oncurrent.desktop
===================================================================
--- apps/konqueror/settings/kio/uasproviders/firefox30oncurrent.desktop	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/konqueror/settings/kio/uasproviders/firefox30oncurrent.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -14,7 +14,7 @@
 Name[eo]=UA-Priskribo (Fajrovulpo 3.0 sub nuna)
 Name[es]=UADescription (Firefox 3.0 en actual)
 Name[et]=UADescription (Selles masinas töötav Firefox 3.0)
-Name[eu]=UADescription (Firefox 3.0 unekoan)
+Name[eu]=UA azalpena (Firefox 3.0 unekoan)
 Name[fi]=Käyttäjäagenttikuvaus (Firefox 3.0 nykyisellä)
 Name[fr]=UADescription (Firefox 3.0 sous le système actuel)
 Name[fy]=UA-beskriuwing (Firefox 3.0 op aktive)
Index: apps/konqueror/sidebar/places_module/konqsidebar_places.desktop
===================================================================
--- apps/konqueror/sidebar/places_module/konqsidebar_places.desktop	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/konqueror/sidebar/places_module/konqsidebar_places.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -13,12 +13,14 @@
 Name[en_GB]=Places SideBar Module
 Name[es]=Módulo de la barra lateral de lugares
 Name[et]=Asukohtade külgriba moodul
+Name[eu]=Lekuen alboko barraren modulua
 Name[fi]=Sijainnit sivupalkki-moduuli
 Name[fr]=Placer le module de barre de navigation 
 Name[fy]=Places sydbalke module
 Name[ga]=Modúl Barra Taoibh Áiteanna
 Name[he]=מודול סרגל צד מקומות
 Name[hi]= बाज़ूपट्टी मॉड्यू लगाता हैल
+Name[hr]=Postavlja modul bočne trake
 Name[ia]=Modulo de Barra Lateral de Placias
 Name[id]=Tempat Modul Batang Sisi
 Name[is]=Staðahliðarsláreining
@@ -34,6 +36,7 @@
 Name[nb]=Modul for steder-sidestolpe
 Name[nds]=Steden-Sietpaneelmoduul
 Name[nl]=Plaatst module voor zijbalk
+Name[nn]=Modul for stad-sidestolpe
 Name[pa]=ਥਾਵਾਂ ਬਾਹਰੀ ਮੋਡੀਊਲ
 Name[pl]=Moduł paska bocznego miejsc
 Name[pt]=Módulo da Barra Lateral dos Locais
Index: apps/konqueror/sidebar/default_entries/places.desktop
===================================================================
--- apps/konqueror/sidebar/default_entries/places.desktop	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/konqueror/sidebar/default_entries/places.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -83,6 +83,7 @@
 Comment[en_GB]=This is the list of places.
 Comment[es]=Esta es la lista de lugares.
 Comment[et]=Asukohtade loend.
+Comment[eu]=Hau da lekuen zerrenda.
 Comment[fi]=Tämä on sijaintien luettelo.
 Comment[fr]=Ceci est la liste des emplacements.
 Comment[fy]=Dit is de opsomming fan places.
@@ -105,6 +106,7 @@
 Comment[nb]=Dette er lista over steder.
 Comment[nds]=Dit is de List mit Steden.
 Comment[nl]=Dit is de lijst met plaatsen.
+Comment[nn]=Dette er lista over stader.
 Comment[pa]=ਇਹ ਥਾਵਾਂ ਦੀ ਲਿਸਟ ਹੈ।
 Comment[pl]=To jest lista miejsc.
 Comment[pt]=Esta é a lista de locais.
Index: apps/konqueror/preloader/konqy_preloader.desktop
===================================================================
--- apps/konqueror/preloader/konqy_preloader.desktop	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/konqueror/preloader/konqy_preloader.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -12,6 +12,7 @@
 Name[en_GB]=Konqueror Browser Preloader
 Name[es]=Módulo de precarga del navegador Konqueror
 Name[et]=Konquerori brauseri eellaadimise moodul
+Name[eu]=Konqueror arakatzailearen aurrezamatzailea
 Name[fi]=Konqueror-selaimen esilataaja
 Name[fr]=Module de préchargement du navigateur Konqueror
 Name[fy]=Konqueror Blêder foarlader
@@ -35,6 +36,7 @@
 Name[nb]=Forhåndslaster for Konqueror nettleser
 Name[nds]=Konqueror vörladen
 Name[nl]=Konqueror browser preloader-module
+Name[nn]=Førehandslastar for nettlesaren Konqueror
 Name[pa]=ਕੋਨਕਿਉਰੋਰ ਬਰਾਊਜ਼ਰ ਪਹਿਲਾਂ ਲੋਡ ਮੋਡੀਊਲ
 Name[pl]=Wstępne wczytywanie dla Konquerora
 Name[pt]=Pré-Carregamento do Navegador Konqueror
Index: apps/dolphin/src/viewextensionsfactory.cpp
===================================================================
--- apps/dolphin/src/viewextensionsfactory.cpp	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/dolphin/src/viewextensionsfactory.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -61,7 +61,7 @@
         m_toolTipManager = new ToolTipManager(view, proxyModel);
 
         connect(dolphinViewController, SIGNAL(hideToolTip()),
-                m_toolTipManager, SLOT(hideTip()));
+                m_toolTipManager, SLOT(hideToolTip()));
     }
 
     // initialize preview generator
Index: apps/dolphin/src/dolphinview.cpp
===================================================================
--- apps/dolphin/src/dolphinview.cpp	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/dolphin/src/dolphinview.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -93,7 +93,6 @@
     m_dolphinViewController(0),
     m_viewModeController(0),
     m_viewAccessor(proxyModel),
-    m_selectionModel(0),
     m_selectionChangedTimer(0),
     m_rootUrl(),
     m_activeItemUrl(),
@@ -540,30 +539,33 @@
 
 void DolphinView::setUrl(const KUrl& url)
 {
-    if (m_viewModeController->url() != url) {
-        m_newFileNames.clear();
+    if (m_viewModeController->url() == url) {
+        return;
+    }
 
-        m_viewModeController->setUrl(url); // emits urlChanged, which we forward
-        m_viewAccessor.prepareUrlChange(url);
-        applyViewProperties();
-        loadDirectory(url);
+    // The selection model might change in the case of the column view. Disconnect
+    // from the current selection model and reconnect later after the URL switch.
+    QAbstractItemView* view = m_viewAccessor.itemView();
+    disconnect(view->selectionModel(), SIGNAL(selectionChanged(QItemSelection, QItemSelection)),
+               this, SLOT(slotSelectionChanged(QItemSelection, QItemSelection)));
 
-        // When changing the URL there is no need to keep the version
-        // data of the previous URL.
-        m_viewAccessor.dirModel()->clearVersionData();
+    m_newFileNames.clear();
 
-        emit startedPathLoading(url);
-    }
+    m_viewModeController->setUrl(url); // emits urlChanged, which we forward
+    m_viewAccessor.prepareUrlChange(url);
+    applyViewProperties();
+    loadDirectory(url);
 
-    // the selection model might have changed in the case of a column view
-    QItemSelectionModel* selectionModel = m_viewAccessor.itemView()->selectionModel();
-    if (m_selectionModel != selectionModel) {
-        disconnect(m_selectionModel, SIGNAL(selectionChanged(QItemSelection, QItemSelection)),
-                   this, SLOT(slotSelectionChanged(QItemSelection, QItemSelection)));
-        m_selectionModel = selectionModel;
-        connect(m_selectionModel, SIGNAL(selectionChanged(QItemSelection, QItemSelection)),
-                this, SLOT(slotSelectionChanged(QItemSelection, QItemSelection)));
-    }
+    // When changing the URL there is no need to keep the version
+    // data of the previous URL.
+    m_viewAccessor.dirModel()->clearVersionData();
+
+    emit startedPathLoading(url);
+
+    // Reconnect to the (probably) new selection model
+    view = m_viewAccessor.itemView();
+    connect(view->selectionModel(), SIGNAL(selectionChanged(QItemSelection, QItemSelection)),
+            this, SLOT(slotSelectionChanged(QItemSelection, QItemSelection)));
 }
 
 void DolphinView::selectAll()
@@ -1271,28 +1273,28 @@
 
 void DolphinView::createView()
 {
+    QAbstractItemView* view = m_viewAccessor.itemView();
+    if ((view != 0) && (view->selectionModel() != 0)) {
+        disconnect(view->selectionModel(), SIGNAL(selectionChanged(QItemSelection, QItemSelection)),
+                   this, SLOT(slotSelectionChanged(QItemSelection, QItemSelection)));
+    }
+
     deleteView();
 
     Q_ASSERT(m_viewAccessor.itemView() == 0);
     m_viewAccessor.createView(this, m_dolphinViewController, m_viewModeController, m_mode);
 
-    QAbstractItemView* view = m_viewAccessor.itemView();
+    view = m_viewAccessor.itemView();
     Q_ASSERT(view != 0);
     view->installEventFilter(this);
     view->viewport()->installEventFilter(this);
 
     m_dolphinViewController->setItemView(view);
 
-    // When changing the view mode, the selection is lost due to reinstantiating
-    // a new item view with a custom selection model. Pass the ownership of the
-    // selection model to DolphinView, so that it can be shared by all item views.
-    if (m_selectionModel != 0) {
-        view->setSelectionModel(m_selectionModel);
-    } else {
-        m_selectionModel = view->selectionModel();
-    }
-    m_selectionModel->setParent(this);
-    connect(m_selectionModel, SIGNAL(selectionChanged(QItemSelection, QItemSelection)),
+    const int zoomLevel = ZoomLevelInfo::zoomLevelForIconSize(view->iconSize());
+    m_viewModeController->setZoomLevel(zoomLevel);
+
+    connect(view->selectionModel(), SIGNAL(selectionChanged(QItemSelection, QItemSelection)),
             this, SLOT(slotSelectionChanged(QItemSelection, QItemSelection)));
 
     setFocusProxy(m_viewAccessor.layoutTarget());
Index: apps/dolphin/src/dolphincolumnviewcontainer.cpp
===================================================================
--- apps/dolphin/src/dolphincolumnviewcontainer.cpp	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/dolphin/src/dolphincolumnviewcontainer.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -108,7 +108,7 @@
 
     int columnIndex = 0;
     foreach (DolphinColumnView* column, m_columns) {
-        if (column->url() == url) {
+        if (column->url().equals(url, KUrl::CompareWithoutTrailingSlash)) {
             // the column represents already the requested URL, hence activate it
             requestActivation(column);
             layoutColumns();
@@ -411,6 +411,8 @@
     } else {
         column->deleteLater();
     }
+
+    layoutColumns();
 }
 
 #include "dolphincolumnviewcontainer.moc"
Index: apps/dolphin/src/tooltips/filemetadatatooltip.h
===================================================================
--- apps/dolphin/src/tooltips/filemetadatatooltip.h	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/dolphin/src/tooltips/filemetadatatooltip.h	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -18,8 +18,8 @@
  *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA            *
  ***************************************************************************/
 
-#ifndef DOLPHINCONTROLLER_H
-#define DOLPHINCONTROLLER_H
+#ifndef FILEMETADATATOOLTIP_H
+#define FILEMETADATATOOLTIP_H
 
 #include <QWidget>
 
@@ -40,14 +40,26 @@
     virtual ~FileMetaDataToolTip();
 
     void setPreview(const QPixmap& pixmap);
-    const QPixmap* preview() const;
+    QPixmap preview() const;
 
     void setName(const QString& name);
     QString name() const;
 
+    /**
+     * Sets the items for which the meta data should be shown.
+     * The signal metaDataRequestFinished() will be emitted,
+     * as soon as the meta data for the items has been received.
+     */
     void setItems(const KFileItemList& items);
     KFileItemList items() const;
 
+signals:
+    /**
+     * Is emitted after the meta data has been received for the items
+     * set by FileMetaDataToolTip::setItems().
+     */
+    void metaDataRequestFinished(const KFileItemList& items);
+
 protected:
     virtual void paintEvent(QPaintEvent* event);
 
Index: apps/dolphin/src/tooltips/tooltipmanager.cpp
===================================================================
--- apps/dolphin/src/tooltips/tooltipmanager.cpp	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/dolphin/src/tooltips/tooltipmanager.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -19,13 +19,9 @@
 
 #include "tooltipmanager.h"
 
-#include "dolphinmodel.h"
-#include "dolphinsortfilterproxymodel.h"
-
 #include "filemetadatatooltip.h"
-#include <kicon.h>
-#include <kio/previewjob.h>
-#include <kwindowsystem.h>
+#include <KIcon>
+#include <KIO/PreviewJob>
 
 #include <QApplication>
 #include <QDesktopWidget>
@@ -33,22 +29,22 @@
 #include <QScrollBar>
 #include <QTimer>
 
+#include <dolphinmodel.h>
+#include <dolphinsortfilterproxymodel.h>
+
 ToolTipManager::ToolTipManager(QAbstractItemView* parent,
                                DolphinSortFilterProxyModel* model) :
     QObject(parent),
     m_view(parent),
     m_dolphinModel(0),
     m_proxyModel(model),
-    m_prepareToolTipTimer(0),
-    m_startPreviewJobTimer(0),
-    m_waitOnPreviewTimer(0),
-    m_showToolTipDelayedTimer(0),
+    m_showToolTipTimer(0),
+    m_contentRetrievalTimer(0),
     m_fileMetaDataToolTip(0),
+    m_toolTipRequested(false),
+    m_metaDataRequested(false),
     m_item(),
-    m_itemRect(),
-    m_generatingPreview(false),
-    m_hasDefaultIcon(false),
-    m_previewPixmap()
+    m_itemRect()
 {
     static FileMetaDataToolTip* sharedToolTip = 0;
     if (sharedToolTip == 0) {
@@ -58,7 +54,9 @@
         // not work.
     }
     m_fileMetaDataToolTip = sharedToolTip;
-    
+    connect(m_fileMetaDataToolTip, SIGNAL(metaDataRequestFinished(KFileItemList)),
+            this, SLOT(slotMetaDataRequestFinished()));
+
     m_dolphinModel = static_cast<DolphinModel*>(m_proxyModel->sourceModel());
     connect(parent, SIGNAL(entered(const QModelIndex&)),
             this, SLOT(requestToolTip(const QModelIndex&)));
@@ -66,33 +64,25 @@
             this, SLOT(hideToolTip()));
 
     // Initialize timers
-    m_prepareToolTipTimer = new QTimer(this);
-    m_prepareToolTipTimer->setSingleShot(true);
-    m_prepareToolTipTimer->setInterval(500);
-    connect(m_prepareToolTipTimer, SIGNAL(timeout()), this, SLOT(prepareToolTip()));   
-    
-    m_startPreviewJobTimer = new QTimer(this);
-    m_startPreviewJobTimer->setSingleShot(true);
-    m_startPreviewJobTimer->setInterval(200);
-    connect(m_startPreviewJobTimer, SIGNAL(timeout()), this, SLOT(startPreviewJob()));
-    
-    m_waitOnPreviewTimer = new QTimer(this);
-    m_waitOnPreviewTimer->setSingleShot(true);
-    m_waitOnPreviewTimer->setInterval(250);     
-    connect(m_waitOnPreviewTimer, SIGNAL(timeout()), this, SLOT(prepareToolTip()));
+    m_showToolTipTimer = new QTimer(this);
+    m_showToolTipTimer->setSingleShot(true);
+    m_showToolTipTimer->setInterval(500);
+    connect(m_showToolTipTimer, SIGNAL(timeout()), this, SLOT(showToolTip()));
 
-    m_showToolTipDelayedTimer = new QTimer(this);
-    m_showToolTipDelayedTimer->setSingleShot(true);
-    m_showToolTipDelayedTimer->setInterval(100);     
-    connect(m_showToolTipDelayedTimer, SIGNAL(timeout()), this, SLOT(showToolTip()));
-    
+    m_contentRetrievalTimer = new QTimer(this);
+    m_contentRetrievalTimer->setSingleShot(true);
+    m_contentRetrievalTimer->setInterval(200);
+    connect(m_contentRetrievalTimer, SIGNAL(timeout()), this, SLOT(startContentRetrieval()));
+
+    Q_ASSERT(m_contentRetrievalTimer->interval() < m_showToolTipTimer->interval());
+
     // When the mousewheel is used, the items don't get a hovered indication
     // (Qt-issue #200665). To assure that the tooltip still gets hidden,
     // the scrollbars are observed.
     connect(parent->horizontalScrollBar(), SIGNAL(valueChanged(int)),
-            this, SLOT(hideTip()));
+            this, SLOT(hideToolTip()));
     connect(parent->verticalScrollBar(), SIGNAL(valueChanged(int)),
-            this, SLOT(hideTip()));
+            this, SLOT(hideToolTip()));
 
     m_view->viewport()->installEventFilter(this);
     m_view->installEventFilter(this);
@@ -102,9 +92,17 @@
 {
 }
 
-void ToolTipManager::hideTip()
+void ToolTipManager::hideToolTip()
 {
-    hideToolTip();
+    QApplication::restoreOverrideCursor();
+
+    m_toolTipRequested = false;
+    m_metaDataRequested = false;
+    m_showToolTipTimer->stop();
+    m_contentRetrievalTimer->stop();
+
+    m_fileMetaDataToolTip->setItems(KFileItemList());
+    m_fileMetaDataToolTip->hide();
 }
 
 bool ToolTipManager::eventFilter(QObject* watched, QEvent* event)
@@ -128,7 +126,7 @@
 void ToolTipManager::requestToolTip(const QModelIndex& index)
 {
     hideToolTip();
-    
+
     // Only request a tooltip for the name column and when no selection or
     // drag & drop operation is done (indicated by the left mouse button)
     if ((index.column() == DolphinModel::Name) && !(QApplication::mouseButtons() & Qt::LeftButton)) {
@@ -139,44 +137,27 @@
         const QModelIndex dirIndex = m_proxyModel->mapToSource(index);
         m_item = m_dolphinModel->itemForIndex(dirIndex);
 
-        // Only start the previewJob when the mouse has been over this item for 200 milliseconds.
-        // This prevents a lot of useless preview jobs when passing rapidly over a lot of items.
-        m_startPreviewJobTimer->start();
-        m_previewPixmap = QPixmap();
-        m_hasDefaultIcon = false;
-
-        m_prepareToolTipTimer->start();
+        // Only start the retrieving of the content, when the mouse has been over this
+        // item for 200 milliseconds. This prevents a lot of useless preview jobs and
+        // meta data retrieval, when passing rapidly over a lot of items.
+        m_contentRetrievalTimer->start();
+        m_showToolTipTimer->start();
+        m_toolTipRequested = true;
+        Q_ASSERT(!m_metaDataRequested);
     }
 }
 
-void ToolTipManager::hideToolTip()
+void ToolTipManager::startContentRetrieval()
 {
-    m_prepareToolTipTimer->stop();
-    m_startPreviewJobTimer->stop();
-    m_waitOnPreviewTimer->stop();
-    m_showToolTipDelayedTimer->stop();
+    if (!m_toolTipRequested) {
+        return;
+    }
 
-    m_fileMetaDataToolTip->hide();
-}
+    m_fileMetaDataToolTip->setName(m_item.text());
 
-void ToolTipManager::prepareToolTip()
-{
-    if (m_generatingPreview) {
-        m_waitOnPreviewTimer->start();
-    }
+    m_metaDataRequested = true;
+    m_fileMetaDataToolTip->setItems(KFileItemList() << m_item);
 
-    if (!m_previewPixmap.isNull()) {
-        showToolTipDelayed(m_previewPixmap);
-    } else if (!m_hasDefaultIcon) {
-        const QPixmap image(KIcon(m_item.iconName()).pixmap(128, 128));
-        showToolTipDelayed(image);
-        m_hasDefaultIcon = true;
-    }
-}
-
-void ToolTipManager::startPreviewJob()
-{
-    m_generatingPreview = true;
     KIO::PreviewJob* job = KIO::filePreview(KFileItemList() << m_item, 256, 256);
 
     connect(job, SIGNAL(gotPreview(const KFileItem&, const QPixmap&)),
@@ -189,44 +170,98 @@
 void ToolTipManager::setPreviewPix(const KFileItem& item,
                                    const QPixmap& pixmap)
 {
-    if ((m_item.url() != item.url()) || pixmap.isNull()) {
-        // An old preview or an invalid preview has been received
+    if (!m_toolTipRequested || (m_item.url() != item.url())) {
+        // No tooltip is requested anymore or an old preview has been received
+        return;
+    }
+
+    if (pixmap.isNull()) {
         previewFailed();
     } else {
-        m_previewPixmap = pixmap;
-        m_generatingPreview = false;
+        m_fileMetaDataToolTip->setPreview(pixmap);
+        if (!m_showToolTipTimer->isActive()) {
+            showToolTip();
+        }
     }
 }
 
 void ToolTipManager::previewFailed()
 {
-    m_generatingPreview = false;
+    if (!m_toolTipRequested) {
+        return;
+    }
+
+    const QPixmap pixmap = KIcon(m_item.iconName()).pixmap(128, 128);
+    m_fileMetaDataToolTip->setPreview(pixmap);
+    if (!m_showToolTipTimer->isActive()) {
+        showToolTip();
+    }
 }
 
+void ToolTipManager::slotMetaDataRequestFinished()
+{
+    if (!m_toolTipRequested) {
+        return;
+    }
+
+    m_metaDataRequested = false;
+    if (!m_showToolTipTimer->isActive()) {
+        showToolTip();
+    }
+}
+
 void ToolTipManager::showToolTip()
 {
-    const QRect desktop = QApplication::desktop()->screenGeometry(m_itemRect.bottomRight());
-    const QSize size = m_fileMetaDataToolTip->sizeHint();
-    
+    Q_ASSERT(m_toolTipRequested);
+    QApplication::restoreOverrideCursor();
+
+    if (QApplication::mouseButtons() & Qt::LeftButton) {
+        return;
+    }
+
+    if (m_fileMetaDataToolTip->preview().isNull() || m_metaDataRequested) {
+        QApplication::setOverrideCursor(QCursor(Qt::WaitCursor));
+        return;
+    }
+
+    const QRect screen = QApplication::desktop()->screenGeometry(QCursor::pos());
+
+    // Restrict tooltip size to current screen size when needed.
+    // Because layout controlling widget doesn't respect widget's maximumSize property
+    // (correct me if I'm wrong), we need to let layout do its work, then manually change
+    // geometry if resulting widget doesn't fit the screen.
+
+    // Step #1 - make sizeHint return calculated tooltip size
+    m_fileMetaDataToolTip->layout()->setSizeConstraint(QLayout::SetFixedSize);
+    m_fileMetaDataToolTip->adjustSize();
+    QSize size = m_fileMetaDataToolTip->sizeHint();
+
+    // Step #2 - correct tooltip size when needed
+    if (size.width() > screen.width()) {
+        size.setWidth(screen.width());
+    }
+    if (size.height() > screen.height()) {
+        size.setHeight(screen.height());
+    }
+
     // m_itemRect defines the area of the item, where the tooltip should be
     // shown. Per default the tooltip is shown in the bottom right corner.
-    // If the tooltip content exceeds the desktop borders, it must be assured that:
+    // It must be assured that:
     // - the content is fully visible
     // - the content is not drawn inside m_itemRect
-    const bool hasRoomToLeft  = (m_itemRect.left()   - size.width()  >= desktop.left());
-    const bool hasRoomToRight = (m_itemRect.right()  + size.width()  <= desktop.right());
-    const bool hasRoomAbove   = (m_itemRect.top()    - size.height() >= desktop.top());
-    const bool hasRoomBelow   = (m_itemRect.bottom() + size.height() <= desktop.bottom());
+    const bool hasRoomToLeft  = (m_itemRect.left()   - size.width()  >= screen.left());
+    const bool hasRoomToRight = (m_itemRect.right()  + size.width()  <= screen.right());
+    const bool hasRoomAbove   = (m_itemRect.top()    - size.height() >= screen.top());
+    const bool hasRoomBelow   = (m_itemRect.bottom() + size.height() <= screen.bottom());
     if (!hasRoomAbove && !hasRoomBelow && !hasRoomToLeft && !hasRoomToRight) {
         return;
     }
-    
-    int x = 0;
-    int y = 0;
+
+    int x, y;
     if (hasRoomBelow || hasRoomAbove) {
         x = QCursor::pos().x() + 16; // TODO: use mouse pointer width instead of the magic value of 16
-        if (x + size.width() >= desktop.right()) {
-            x = desktop.right() - size.width();
+        if (x + size.width() >= screen.right()) {
+            x = screen.right() - size.width() + 1;
         }
         if (hasRoomBelow) {
             y = m_itemRect.bottom() + 1;
@@ -240,38 +275,18 @@
         } else {
             x = m_itemRect.left() - size.width();
         }
-
         // Put the tooltip at the bottom of the screen. The x-coordinate has already
         // been adjusted, so that no overlapping with m_itemRect occurs.
-        y = desktop.bottom() - size.height();
+        y = screen.bottom() - size.height() + 1;
     }
-    
-    m_fileMetaDataToolTip->move(qMax(x, 0), qMax(y, 0));
+
+    // Step #3 - Alter tooltip geometry
+    m_fileMetaDataToolTip->setFixedSize(size);
+    m_fileMetaDataToolTip->layout()->setSizeConstraint(QLayout::SetNoConstraint);
+    m_fileMetaDataToolTip->setGeometry(x, y, size.width(), size.height());
     m_fileMetaDataToolTip->show();
-}
 
-void ToolTipManager::showToolTipDelayed(const QPixmap& pixmap)
-{
-    if (QApplication::mouseButtons() & Qt::LeftButton) {
-        return;
-    }
-    
-    m_fileMetaDataToolTip->setPreview(pixmap);
-    m_fileMetaDataToolTip->setName(m_item.text());
-    m_fileMetaDataToolTip->setItems(KFileItemList() << m_item);
-
-    // Because one tooltip instance is reused, the size hint always depends on the previous
-    // content (QWidgets don't update their layout geometry if they are invisible). To
-    // assure having a consistent size without relayout flickering, the tooltip is opened
-    // on an invisible position first. This gives the layout system some time to asynchronously
-    // update the content. Sadly this only works with compositing enabled.
-    if (KWindowSystem::compositingActive()) {
-        const QRect desktop = QApplication::desktop()->screenGeometry(m_itemRect.bottomRight());
-        m_fileMetaDataToolTip->move(desktop.bottomRight());
-        m_fileMetaDataToolTip->show();
-    }
-    
-    m_showToolTipDelayedTimer->start(); // Calls ToolTipManager::showToolTip()
+    m_toolTipRequested = false;
 }
 
 #include "tooltipmanager.moc"
Index: apps/dolphin/src/tooltips/tooltipmanager.h
===================================================================
--- apps/dolphin/src/tooltips/tooltipmanager.h	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/dolphin/src/tooltips/tooltipmanager.h	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -23,7 +23,7 @@
 #include <QObject>
 #include <QRect>
 
-#include <kfileitem.h>
+#include <KFileItem>
 
 class DolphinModel;
 class DolphinSortFilterProxyModel;
@@ -54,53 +54,38 @@
      * only needed when the tooltip should be hidden although
      * an item is hovered.
      */
-    void hideTip();
+    void hideToolTip();
 
 protected:
     virtual bool eventFilter(QObject* watched, QEvent* event);
 
 private slots:
     void requestToolTip(const QModelIndex& index);
-    void hideToolTip();
-    void prepareToolTip();
-    void startPreviewJob();
+    void startContentRetrieval();
     void setPreviewPix(const KFileItem& item, const QPixmap& pix);
     void previewFailed();
+    void slotMetaDataRequestFinished();
     void showToolTip();
-    
-private:
-    void showToolTipDelayed(const QPixmap& pixmap);
 
 private:
     QAbstractItemView* m_view;
     DolphinModel* m_dolphinModel;
     DolphinSortFilterProxyModel* m_proxyModel;
 
-    /// Timeout from requesting a tooltip until the tooltip is shown
-    QTimer* m_prepareToolTipTimer;
-    
-    /// Timeout from requesting a tooltip until starting a job to
-    /// create a preview pixmap. The preview job is started before
-    /// m_prepareToolTipTimer has been exceeded, to have the preview
-    /// pixmap ideally before the tooltip will be shown.
-    QTimer* m_startPreviewJobTimer;
-    
-    /// Don't show the tooltip, before the preview has been received. The
-    /// time indicates the interval, when the check for a received
-    /// is done.
-    QTimer* m_waitOnPreviewTimer;
-    
-    /// The tooltip is shown slightly delayed to prevent a flickering
-    /// because of layouting the content.
-    QTimer* m_showToolTipDelayedTimer;
+    /// Timeout from requesting a tooltip until the tooltip
+    /// should be shown
+    QTimer* m_showToolTipTimer;
 
+    /// Timeout from requesting a tooltip until the retrieving of
+    /// the tooltip content like preview and meta data gets started.
+    QTimer* m_contentRetrievalTimer;
+
     FileMetaDataToolTip* m_fileMetaDataToolTip;
 
+    bool m_toolTipRequested;
+    bool m_metaDataRequested;
     KFileItem m_item;
     QRect m_itemRect;
-    bool m_generatingPreview;
-    bool m_hasDefaultIcon;
-    QPixmap m_previewPixmap;
 };
 
 #endif
Index: apps/dolphin/src/tooltips/filemetadatatooltip.cpp
===================================================================
--- apps/dolphin/src/tooltips/filemetadatatooltip.cpp	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/dolphin/src/tooltips/filemetadatatooltip.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -20,10 +20,10 @@
 
 #include "filemetadatatooltip.h"
 
-#include <kcolorscheme.h>
+#include <KColorScheme>
 #include <kfilemetadatawidget.h>
-#include <kseparator.h>
-#include <kwindowsystem.h>
+#include <KSeparator>
+#include <KWindowSystem>
 
 #include <QLabel>
 #include <QPainter>
@@ -34,7 +34,6 @@
     m_preview(0),
     m_name(0),
     m_fileMetaDataWidget(0)
-
 {
     setAttribute(Qt::WA_TranslucentBackground);
     setWindowFlags(Qt::ToolTip | Qt::FramelessWindowHint);
@@ -47,30 +46,25 @@
     QFont font = m_name->font();
     font.setBold(true);
     m_name->setFont(font);
-    m_name->setAlignment(Qt::AlignHCenter);
 
     // Create widget for the meta data
     m_fileMetaDataWidget = new KFileMetaDataWidget();
     m_fileMetaDataWidget->setForegroundRole(QPalette::ToolTipText);
     m_fileMetaDataWidget->setReadOnly(true);
+    connect(m_fileMetaDataWidget, SIGNAL(metaDataRequestFinished(KFileItemList)),
+            this, SIGNAL(metaDataRequestFinished(KFileItemList)));
 
-    // The stretchwidget allows the metadata widget to be top aligned and fills
-    // the remaining vertical space
-    QWidget* stretchWidget = new QWidget(this);
-    stretchWidget->setSizePolicy(QSizePolicy::Preferred, QSizePolicy::MinimumExpanding);
-
-    QWidget* textContainer = new QWidget(this);
-    QVBoxLayout* textLayout = new QVBoxLayout(textContainer);
+    QVBoxLayout* textLayout = new QVBoxLayout();
+    textLayout->setAlignment(Qt::AlignTop);
     textLayout->addWidget(m_name);
     textLayout->addWidget(new KSeparator());
     textLayout->addWidget(m_fileMetaDataWidget);
-    textLayout->addWidget(stretchWidget);
+    textLayout->setAlignment(m_name, Qt::AlignCenter);
+    textLayout->setAlignment(m_fileMetaDataWidget, Qt::AlignLeft);
 
     QHBoxLayout* tipLayout = new QHBoxLayout(this);
     tipLayout->addWidget(m_preview);
-    tipLayout->addWidget(textContainer);
-    
-    tipLayout->setSizeConstraint(QLayout::SetFixedSize);
+    tipLayout->addLayout(textLayout);
 }
 
 FileMetaDataToolTip::~FileMetaDataToolTip()
@@ -82,9 +76,12 @@
     m_preview->setPixmap(pixmap);
 }
 
-const QPixmap* FileMetaDataToolTip::preview() const
+QPixmap FileMetaDataToolTip::preview() const
 {
-    return m_preview->pixmap();
+    if (m_preview->pixmap() != 0) {
+        return *m_preview->pixmap();
+    }
+    return QPixmap();
 }
 
 void FileMetaDataToolTip::setName(const QString& name)
Index: apps/dolphin/src/dolphinview.h
===================================================================
--- apps/dolphin/src/dolphinview.h	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/dolphin/src/dolphinview.h	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -796,7 +796,6 @@
     ViewModeController* m_viewModeController;
     ViewAccessor m_viewAccessor;
 
-    QItemSelectionModel* m_selectionModel; // allow to switch views without losing the selection
     QTimer* m_selectionChangedTimer;
 
     KUrl m_rootUrl;
Index: apps/lib/konq/Templates/Floppy.desktop
===================================================================
--- apps/lib/konq/Templates/Floppy.desktop	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/lib/konq/Templates/Floppy.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -13,11 +13,13 @@
 Name[en_GB]=Format
 Name[es]=Formato
 Name[et]=Vormindamine
+Name[eu]=Formatua
 Name[fi]=Alusta
 Name[fr]=Format
 Name[fy]=Opmaak
 Name[ga]=Formáidigh
 Name[hi]=फॉर्मेट
+Name[hr]=Formatiranje
 Name[ia]=Format
 Name[id]=Format
 Name[is]=Forsníða
@@ -33,6 +35,7 @@
 Name[nb]=Format
 Name[nds]=Formateren
 Name[nl]=Formatteren
+Name[nn]=Formater
 Name[pa]=ਫਾਰਮੈਟ
 Name[pl]=Format
 Name[pt]=Formatar
Index: apps/lib/konq/favicons/favicons.desktop
===================================================================
--- apps/lib/konq/favicons/favicons.desktop	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/lib/konq/favicons/favicons.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -82,6 +82,7 @@
 Comment[en_GB]=Stores website icons
 Comment[es]=Almacena iconos de sitios web
 Comment[et]=Veebilehekülgede ikoonide salvestamine
+Comment[eu]=Webguneen ikonoak gordetzen ditu
 Comment[fi]=Säilyttää nettisivujen kuvakkeita
 Comment[fr]=Stocker les icônes des site Internet
 Comment[fy]=Bewarret website byldkaikes
@@ -103,6 +104,7 @@
 Comment[nb]=Lagrer nettstedsikoner
 Comment[nds]=Wohrt Nettsiedenlüttbiller
 Comment[nl]=Slaat website pictogrammen op
+Comment[nn]=Lagrar nettstadikon
 Comment[pa]=ਵੈੱਬਸਾਈਟ ਆਈਕਾਨ ਸਟੋਰ ਕਰਦਾ ਹੈ
 Comment[pl]=Przechowuje ikony stron WWW
 Comment[pt]=Guarda os ícones das páginas Web
Index: apps/keditbookmarks/keditbookmarks.desktop
===================================================================
--- apps/keditbookmarks/keditbookmarks.desktop	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/keditbookmarks/keditbookmarks.desktop	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -16,6 +16,7 @@
 Name[en_GB]=Bookmark Editor
 Name[es]=Editor de marcadores
 Name[et]=Järjehoidjate redaktor
+Name[eu]=Laster-marka editorea
 Name[fi]=Kirjanmerkkimuokkain
 Name[fr]=Éditeur de signets
 Name[fy]=Blêdwizerbewurker
@@ -38,6 +39,7 @@
 Name[nb]=Bokmerkeredigerer
 Name[nds]=Leesteken-Editor
 Name[nl]=Bladwijzerbewerker
+Name[nn]=Bokmerkeredigering
 Name[pa]=ਬੁੱਕਮਾਰਕ ਐਡੀਟਰ
 Name[pl]=Edytor zakładek
 Name[pt]=Editor de Favoritos
@@ -71,6 +73,7 @@
 Comment[en_GB]=Bookmark Organiser and Editor
 Comment[es]=Organizador y editor de marcadores
 Comment[et]=Järjehoidjate korraldaja ja redaktor
+Comment[eu]=Laster-marka antolatzaile eta editorea
 Comment[fi]=Kirjanmerkkimuokkain ja -järjestelijä
 Comment[fr]=Éditeur et organisateur de signets
 Comment[fy]=Blêdwizers organisearje en bewurker
@@ -92,6 +95,7 @@
 Comment[nb]=Bokmerkeordner og redigerer
 Comment[nds]=Leestekenpleger un -editor
 Comment[nl]=Bladwijzers organiseren en bewerken
+Comment[nn]=Bokmerkeordnar og redigering
 Comment[pa]=ਬੁੱਕਮਾਰਕ ਪਰਬੰਧਕ ਤੇ ਐਡੀਟਰ
 Comment[pl]=Organizacja i modyfikacja zakładek
 Comment[pt]=Organizador e Editor de Favoritos
Index: apps/keditbookmarks/kbookmarkmodel/model.cpp
===================================================================
--- apps/keditbookmarks/kbookmarkmodel/model.cpp	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/keditbookmarks/kbookmarkmodel/model.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -33,8 +33,6 @@
 #include <QtCore/QStringList>
 #include <QtCore/QMimeData>
 
-Q_DECLARE_METATYPE(KBookmark)
-
 class KBookmarkModel::Private
 {
 public:
Index: apps/keditbookmarks/bookmarklistview.cpp
===================================================================
--- apps/keditbookmarks/bookmarklistview.cpp	(.../tags/KDE/4.5.0/kdebase)	(wersja 1165575)
+++ apps/keditbookmarks/bookmarklistview.cpp	(.../branches/KDE/4.5/kdebase)	(wersja 1165575)
@@ -200,8 +200,6 @@
     return (source_column == 0);
 }
 
-Q_DECLARE_METATYPE(KBookmark)
-
 bool BookmarkFolderViewFilterModel::filterAcceptsRow ( int source_row, const QModelIndex & source_parent ) const
 {
     QModelIndex index = sourceModel()->index(source_row, 0, source_parent);
Index: runtime/knotify/kde.notifyrc
===================================================================
Index: runtime/knotify/hardwarenotifications.notifyrc
===================================================================
Index: runtime/solid-device-automounter/kded/device_automounter.desktop
===================================================================
Index: runtime/kcontrol/knotify/kcmnotify.desktop
===================================================================
Index: runtime/desktoptheme/air/translucent/widgets/panel-background.svgz
===================================================================
Nie można wyświetlić: plik binarny.
svn:mime-type = application/octet-stream
Index: runtime/phonon/platform_kde/phonon.notifyrc
===================================================================
Index: runtime/phonon/kcm/kcm_phonon.desktop
===================================================================
Index: runtime/phonon/kded-module/phononserver.desktop
===================================================================
Index: runtime/kpasswdserver/kpasswdserver.desktop
===================================================================
Index: runtime/khelpcenter/plugins/onlinehelp.desktop
===================================================================
Index: runtime/doc/kioslave/CMakeLists.txt
===================================================================
Index: runtime/kwalletd/kwalletd.notifyrc
===================================================================
Index: runtime/ktimezoned/ktimezoned.desktop
===================================================================
Index: runtime/kioslave/archive/zip.protocol
===================================================================
Index: runtime/kioslave/archive/tar.protocol
===================================================================
Index: runtime/kioslave/thumbnail/imagethumbnail.desktop
===================================================================
Index: runtime/kioslave/filter/xz.protocol
===================================================================
Index: runtime/kioslave/filter/lzma.protocol
===================================================================
Index: runtime/kioslave/network/network/builder/upnp/upnpnetworkbuilder.cpp
===================================================================
Index: runtime/kioslave/network/network/builder/upnp/upnpnetworkbuilder.h
===================================================================
Index: runtime/kioslave/remote/kdedmodule/remotedirnotify.desktop
===================================================================
Index: runtime/kioslave/desktop/desktopnotifier.desktop
===================================================================
Index: runtime/nepomuk/services/removablestorage/nepomukremovablestorageservice.desktop
===================================================================
Index: runtime/nepomuk/services/strigi/nepomukstrigiservice.notifyrc
===================================================================
Index: runtime/nepomuk/services/activities/nepomukactivitiesservice.desktop
===================================================================
Index: runtime/nepomuk/services/storage/nepomukstorage.notifyrc
===================================================================
Index: runtime/nepomuk/services/queryservice/nepomukqueryservice.desktop
===================================================================
Index: runtime/nepomuk/server/nepomukserver.desktop
===================================================================
Index: runtime/solidautoeject/solidautoeject.desktop
===================================================================
Index: runtime/activitymanager/activitymanager.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/facebook.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/github.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/qt.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/identica_people.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/blip.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/beolingus.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/duckduckgo_shopping.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/google_maps.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/kde_forums.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/kde_techbase.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/ecosia.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/magnatune.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/flickrcc.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/yahoo_image.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/yahoo_shopping.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/kde_look.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/yahoo_local.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/youtube.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/urbandictionary.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/amazon_mp3.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/yahoo_video.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/google_code.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/flickr.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/imdb.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/identica_groups.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/duckduckgo.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/php.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/opendesktop.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/kde_userbase.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/7digital.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/wikia.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/jamendo.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/vimeo.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/yahoo.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/gitorious.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/amazon.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/duckduckgo_info.desktop
===================================================================
Index: runtime/kurifilter-plugins/ikws/searchproviders/identica_notices.desktop
===================================================================
Index: runtime/soliduiserver/soliduiserver.desktop
===================================================================
Index: runtime/kglobalaccel/kglobalaccel.notifyrc
===================================================================
Index: runtime/attica/kcm/kcm_attica.desktop
===================================================================
Index: runtime/plasma/tools/plasmapkg/main.cpp
===================================================================
Index: runtime/plasma/remotewidgetshelper/kcm_remotewidgets.actions
===================================================================
Index: workspace/ksmserver/kcm/kcmsmserver.desktop
===================================================================
Index: workspace/ConfigureChecks.cmake
===================================================================
Index: workspace/kcontrol/dateandtime/kcmclock_actions.actions
===================================================================
Index: workspace/kcontrol/desktoptheme/desktoptheme.desktop
===================================================================
Index: workspace/kcontrol/keyboard/xkb_rules.cpp
===================================================================
Index: workspace/kcontrol/keyboard/layout_memory.cpp
===================================================================
Index: workspace/kcontrol/keyboard/kcm_view_models.cpp
===================================================================
Index: workspace/kcontrol/keyboard/keyboard_daemon.cpp
===================================================================
Index: workspace/kcontrol/keyboard/plasma_applet_keyboard.desktop
===================================================================
Index: workspace/kcontrol/keyboard/CMakeLists.txt
===================================================================
Index: workspace/kcontrol/keyboard/xkb_rules.h
===================================================================
Index: workspace/kcontrol/keyboard/layout_memory.h
===================================================================
Index: workspace/kcontrol/keyboard/layout_widget.cpp
===================================================================
Index: workspace/kcontrol/keyboard/xinput_helper.cpp
===================================================================
Index: workspace/kcontrol/keyboard/keyboard.desktop
===================================================================
Index: workspace/kcontrol/keyboard/kcm_keyboard_widget.cpp
===================================================================
Index: workspace/kcontrol/randr/module/randrmonitor.desktop
===================================================================
Index: workspace/kcontrol/workspaceoptions/workspaceoptions.desktop
===================================================================
Index: workspace/kcontrol/kdm/kdm.desktop
===================================================================
Index: workspace/kcontrol/kdm/kcmkdm_actions.actions
===================================================================
Index: workspace/startkde.cmake
===================================================================
Index: workspace/statusnotifierwatcher/statusnotifierwatcher.desktop
===================================================================
Index: workspace/doc/kmenuedit/kmenueditmain.png
===================================================================
Nie można wyświetlić: plik binarny.
svn:mime-type = application/octet-stream
Index: workspace/systemsettings/categories/settings-permissions.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-account-details.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-input-devices.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-workspace-appearance-and-behavior.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-lost-and-found.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-application-and-system-notifications.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-application-appearance-and-behavior.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-locale.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-application-appearance.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-network-and-connectivity.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-desktop-appearance.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-personal-information.desktop
===================================================================
Index: workspace/systemsettings/categories/settings-system-administration.desktop
===================================================================
Index: workspace/kwrited/kwrited.desktop
===================================================================
Index: workspace/wallpapers/Blue_Wood/metadata.desktop
===================================================================
Index: workspace/wallpapers/Grass/metadata.desktop
===================================================================
Index: workspace/wallpapers/Hanami/metadata.desktop
===================================================================
Index: workspace/wallpapers/Fresh_Morning/metadata.desktop
===================================================================
Index: workspace/wallpapers/Autumn/metadata.desktop
===================================================================
Index: workspace/wallpapers/Quadros/metadata.desktop
===================================================================
Index: workspace/wallpapers/Media_Life/metadata.desktop
===================================================================
Index: workspace/solid/networking/kded/networkstatus.desktop
===================================================================
Index: workspace/solid/kcm/kcm_solid.desktop
===================================================================
Index: workspace/CMakeLists.txt
===================================================================
Index: workspace/powerdevil/daemon/powerdevil.desktop
===================================================================
Index: workspace/powerdevil/powerdevil.notifyrc
===================================================================
Index: workspace/krunner/interfaces/default/interface.cpp
===================================================================
Index: workspace/krunner/interfaces/default/interface.h
===================================================================
Index: workspace/krunner/lock/kscreenlocker.notifyrc
===================================================================
Index: workspace/libs/plasmagenericshell/scripting/containment.cpp
===================================================================
Index: workspace/libs/plasmagenericshell/scripting/widget.cpp
===================================================================
Index: workspace/libs/plasmagenericshell/scripting/applet.cpp
===================================================================
Index: workspace/libs/plasmagenericshell/scripting/applet.h
===================================================================
Index: workspace/libs/kworkspace/kactivityinfo.cpp
===================================================================
Index: workspace/libs/kworkspace/CMakeLists.txt
===================================================================
Index: workspace/libs/oxygen/oxygenhelper.cpp
===================================================================
Index: workspace/libs/ksysguard/scripts/smaps/smaps.desktop
===================================================================
Index: workspace/libs/taskmanager/taskgroup.cpp
===================================================================
Index: workspace/freespacenotifier/freespacenotifier.desktop
===================================================================
Index: workspace/freespacenotifier/freespacenotifier.notifyrc
===================================================================
Index: workspace/kstyles/themes/light-v2.themerc
===================================================================
Index: workspace/kstyles/themes/light-v3.themerc
===================================================================
Index: workspace/kstyles/oxygen/animations/oxygenmenubardata.cpp
===================================================================
Index: workspace/kstyles/oxygen/animations/oxygenprogressbardata.cpp
===================================================================
Index: workspace/kstyles/oxygen/oxygenstyle.cpp
===================================================================
Index: workspace/khotkeys/app/khotkeys.desktop
===================================================================
Index: workspace/khotkeys/kcm_hotkeys/khotkeys.desktop
===================================================================
Index: workspace/khotkeys/data/defaults.khotkeys
===================================================================
Index: workspace/khotkeys/data/kde32b1.khotkeys
===================================================================
Index: workspace/khotkeys/data/printscreen.khotkeys
===================================================================
Index: workspace/config-workspace.h.cmake
===================================================================
Index: workspace/kinfocenter/Modules/samba/smbstatus.desktop
===================================================================
Index: workspace/kinfocenter/Modules/infosummary/kcm_infosummary.desktop
===================================================================
Index: workspace/kinfocenter/Modules/devinfo/devinfo.desktop
===================================================================
Index: workspace/kinfocenter/Modules/nics/nic.desktop
===================================================================
Index: workspace/kwin/effects/glide/glide_config.desktop
===================================================================
Index: workspace/kwin/effects/glide/glide.desktop
===================================================================
Index: workspace/kwin/tabbox.cpp
===================================================================
Index: workspace/kwin/kcmkwin/kwindecoration/kwindecoration.desktop
===================================================================
Index: workspace/kwin/kcmkwin/kwindesktop/desktop.desktop
===================================================================
Index: workspace/kwin/kcmkwin/kwinrules/kwinrules.desktop
===================================================================
Index: workspace/kwin/kcmkwin/kwintabbox/kwintabbox.desktop
===================================================================
Index: workspace/kwin/kwin.notifyrc
===================================================================
Index: workspace/kwin/geometry.cpp
===================================================================
Index: workspace/kwin/manage.cpp
===================================================================
Index: workspace/kdm/kfrontend/pics/stripes.png.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/ratpoison.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/wmaker.desktop
===================================================================
Index: workspace/kdm/kfrontend/sessions/lxde.desktop
===================================================================
Index: workspace/ksysguard/gui/ksysguard.desktop
===================================================================
Index: workspace/ksysguard/example/ksysguard.notifyrc
===================================================================
Index: workspace/plasma/netbook/containments/sal/itemcontainer.cpp
===================================================================
Index: workspace/plasma/netbook/containments/sal/sal.cpp
===================================================================
Index: workspace/plasma/netbook/containments/newspaper/appletsview.cpp
===================================================================
Index: workspace/plasma/netbook/containments/newspaper/plasma-containment-newspaper.desktop
===================================================================
Index: workspace/plasma/netbook/containments/newspaper/newspaper.cpp
===================================================================
Index: workspace/plasma/netbook/shell/data/layouts/plasma-layout-org.kde.plasma-netbook.defaultPage.desktop
===================================================================
Index: workspace/plasma/netbook/shell/data/layouts/org.kde.plasma-netbook.defaultPage/metadata.desktop
===================================================================
Index: workspace/plasma/netbook/shell/data/layouts/plasma-layout-org.kde.plasma-netbook.defaultSal.desktop
===================================================================
Index: workspace/plasma/netbook/shell/data/layouts/org.kde.plasma-netbook.defaultSal/metadata.desktop
===================================================================
Index: workspace/plasma/desktop/shell/panelappletoverlay.cpp
===================================================================
Index: workspace/plasma/generic/dataengines/devicenotifications/plasma-dataengine-devicenotifications.desktop
===================================================================
Index: workspace/plasma/generic/dataengines/systemmonitor/plasma-dataengine-systemmonitor.desktop
===================================================================
Index: workspace/plasma/generic/dataengines/weather/ions/wetter.com/ion_wettercom.h
===================================================================
Index: workspace/plasma/generic/dataengines/statusnotifieritem/statusnotifieritemsource.cpp
===================================================================
Index: workspace/plasma/generic/dataengines/statusnotifieritem/plasma_engine_statusnotifieritem.desktop
===================================================================
Index: workspace/plasma/generic/dataengines/statusnotifieritem/CMakeLists.txt
===================================================================
Index: workspace/plasma/generic/applets/system-monitor/plasma-applet-system-monitor.desktop
===================================================================
Index: workspace/plasma/generic/applets/system-monitor/plasma-applet-sm_cpu.desktop
===================================================================
Index: workspace/plasma/generic/applets/system-monitor/plasma-applet-sm_hwinfo.desktop
===================================================================
Index: workspace/plasma/generic/applets/system-monitor/plasma-applet-sm_temperature.desktop
===================================================================
Index: workspace/plasma/generic/applets/system-monitor/plasma-applet-sm_ram.desktop
===================================================================
Index: workspace/plasma/generic/applets/system-monitor/plasma-applet-sm_net.desktop
===================================================================
Index: workspace/plasma/generic/applets/system-monitor/plasma-applet-sm_hdd.desktop
===================================================================
Index: workspace/plasma/generic/applets/battery/plasma-battery-default.desktop
===================================================================
Index: workspace/plasma/generic/applets/battery/battery.cpp
===================================================================
Index: workspace/plasma/generic/runners/services/plasma-runner-services.desktop
===================================================================
Index: workspace/plasma/generic/runners/nepomuksearch/nepomuksearchrunner.cpp
===================================================================
Index: workspace/plasma/generic/runners/nepomuksearch/queryclientwrapper.cpp
===================================================================
Index: workspace/plasma/generic/runners/nepomuksearch/CMakeLists.txt
===================================================================
Index: workspace/plasma/generic/runners/windowedwidgets/plasma-runner-windowedwidgets.desktop
===================================================================
